<!DOCTYPE html>
<!-- saved from url=(0079)https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes -->
<html lang="en" class="pb-page rduajum idc0_326 fonts-loaded fonts-loaded" data-request-id="167fc67e-f862-4922-babf-c30c5118f829"><head data-pb-dropzone="head"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script type="text/javascript">
    (function() {
      var container = document.querySelector('#_tcx-q9dxxv7o0y');
      if (!container) {
        return;
      }

      function addToContainer(url, text) {
        const wrapper = document.createElement('div');
        wrapper.setAttribute('data-tcx-url', url);
        wrapper.innerText = text;
        container.appendChild(wrapper);
      }

      const fetch = window.fetch
      window.fetch = function() {
        return Promise.resolve(fetch.apply(window, arguments))
            .then(async response => {
              if (response.ok) {
                try {
                  const clone = response.clone();
                  const json = await clone.json();
                  addToContainer(clone.url, JSON.stringify(json));
                } catch (err) {}
              }
              return response;
            });
      };

      var XHR = XMLHttpRequest.prototype;
      var send = XHR.send;
      var open = XHR.open;
      XHR.open = function(method, url) {
        this.url = url;
        return open.apply(this, arguments);
      };
      XHR.send = function() {
        this.addEventListener('load', function() {
          try {
            const response = this.response;
            if (response && response.length) {
              const firstChar = response[0];
              if (firstChar === '[' || firstChar === '{') {
                addToContainer(this.url, response);
              }
            }
          } catch (err) {
            // No-op.
          }
        });
        return send.apply(this, arguments);
      };
    })();
  </script><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0966842X20302717;requestedJournal:journal:timi;pageGroup:string:Article View;page:string:Show Full Text;wgroup:string:Default Website Group;journal:journal:timi;csubtype:string:Ahead of Print">




<!-- if there is any customization for Responsive Project widget  -->


























        <meta name="robots" content="noodp">
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">









        <meta property="og:title" content="The Next Million Names for Archaea and Bacteria">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/trends/microbiology/abstract/S0966-842X(20)30271-7">
    



        <meta property="og:site_name" content="Trends in Microbiology">
    


        <meta property="og:image" content="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/asset/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg">
    


        <meta property="og:description" content="Latin binomials, popularised in the 18th century by the Swedish naturalist Linnaeus,
have stood the test of time in providing a stable, clear, and memorable system of
nomenclature across biology. However, relentless and ever-deeper exploration and analysis
of the microbial world has created an urgent need for huge numbers of new names for
Archaea and Bacteria. Manual creation of such names remains difficult and slow and
typically relies on expert-driven nomenclatural quality control. Keen to ensure that
the legacy of Linnaeus lives on in the age of microbial genomics and metagenomics,
we propose an automated approach, employing combinatorial concatenation of roots from
Latin and Greek to create linguistically correct names for genera and species that
can be used off the shelf as needed.">
    







        <meta name="twitter:title" content="The Next Million Names for Archaea and Bacteria">
    




        <meta name="twitter:image" content="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/asset/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg">
    


        <meta name="twitter:description" content="Latin binomials, popularised in the 18th century by the Swedish naturalist Linnaeus,
have stood the test of time in providing a stable, clear, and memorable system of
nomenclature across biology. However, relentless and ever-deeper exploration and analysis
of the microbial world has created an urgent need for huge numbers of new names for
Archaea and Bacteria. Manual creation of such names remains difficult and slow and
typically relies on expert-driven nomenclatural quality control. Keen to ensure that
the legacy of Linnaeus lives on in the age of microbial genomics and metagenomics,
we propose an automated approach, employing combinatorial concatenation of roots from
Latin and Greek to create linguistically correct names for genera and species that
can be used off the shelf as needed.">
    



    <meta name="apple-itunes-app" content="app-id=591217684">


    <meta name="google-play-app" content="app-id=com.cell.journalreader">







    
    

<title>The Next Million Names for Archaea and Bacteria: Trends in Microbiology</title><meta name="citation_journal_title" content="Trends in Microbiology"> <meta name="citation_journal_abbrev" content="Trends in Microbiology"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="The Next Million Names for Archaea and Bacteria"> <meta name="citation_online_date" content="2020/12/04"> <meta name="citation_volume" content="0"> <meta name="citation_issue" content="0"> <meta name="citation_pmid" content="33288384"> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S0966842X20302717/abstract"> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S0966842X20302717/fulltext"> <meta name="citation_pdf_url" content="http://www.cell.com/article/S0966842X20302717/pdf"> <meta name="citation_issn" content="0966-842X"> <meta name="citation_issn" content="1878-4380"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.tim.2020.10.009"> <meta name="citation_pmid" content="33288384"> <meta name="citation_author" content="Mark J. Pallen"> <meta name="citation_author_email" content="mark.pallen@quadram.ac.uk"> <meta name="citation_author_institution" content="Quadram Institute Bioscience, Norwich Research Park, Norwich, UK"> <meta name="citation_author_institution" content="School of Veterinary Medicine, University of Surrey, Guildford, Surrey, UK"> <meta name="citation_author_institution" content="University of East Anglia, Norwich Research Park, Norwich, UK"> <meta name="citation_author" content="Andrea Telatin"> <meta name="citation_author_institution" content="Quadram Institute Bioscience, Norwich Research Park, Norwich, UK"> <meta name="citation_author" content="Aharon Oren"> <meta name="citation_author_institution" content="The Institute of Life Sciences, Edmond J. Safra Campus, The Hebrew University of Jerusalem, Jerusalem, Israel"> <meta name="citation_keywords" content="nomenclature; Candidatus; metagenome-assembled genomes; genome-based taxonomy"> <meta name="citation_reference" content="citation_title=Systema Naturae; citation_author=C. Linnaeus; citation_publication_date=1759; citation_publisher=Laurentius Salvius; "> <meta name="citation_reference" content="citation_title=Animalcula infusoria fluviatilia et marina, quae detexit, systematice descripsit et ad vivum delineari curavit; citation_author=O.F. Müller; citation_publication_date=1786; citation_publisher=Typis Nicolai Mölleri; "> <meta name="citation_reference" content="citation_title=On the Origin of Species; citation_author=C. Darwin; citation_publication_date=1859; citation_publisher=John Murray; "> <meta name="citation_reference" content="citation_title=Grundzüge einer Theorie der phylogenetischen Systematik; citation_author=W. Hennig; citation_publication_date=1950; citation_publisher=Deutscher Zentralverlag; "> <meta name="citation_reference" content="citation_title=International Code of Nomenclature of Prokaryotes; citation_author=C.T. Parker; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=69; citation_firstpage=S1; citation_lastpage=S111; citation_publication_date=2019; "> <meta name="citation_reference" content="citation_title=List of prokaryotic names with standing in nomenclature (LPSN) moves to the DSMZ; citation_author=A.C. Parte; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=All genera of the world: an overview and estimates based on the March 2020 release of the Interim Register of Marine and Nonmarine Genera (IRMNG); citation_author=T. Rees; citation_author=et al.; citation_journal_title=Megataxa; citation_volume=1; citation_firstpage=123; citation_lastpage=140; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=Santa Rosalia revisited: why are there so many species of bacteria; citation_author=D.E. Dykhuizen; citation_journal_title=Antonie van Leeuwenhoek; citation_volume=73; citation_firstpage=25; citation_lastpage=33; citation_publication_date=1998; "> <meta name="citation_reference" content="citation_title=After all, only millions; citation_author=R. Amann; citation_author=R. Rosselló-Móra; citation_journal_title=mBio; citation_volume=7; citation_publication_date=2016; "> <meta name="citation_reference" content="citation_title=To the memory of my beloved, The Author Mr William Shakespeare: And what he hath left us; citation_inbook_title=Mr. William Shakespeares Comedies Histories &amp; Tragedies: Published according to the True Originall Copies; citation_author=B. Jonson; citation_publication_date=1623; citation_publisher=Isaac Iaggard and Edward Blount; "> <meta name="citation_reference" content="citation_title=Nomenclatural Literacy; citation_inbook_title=Handbook of new bacterial systematics; citation_author=T.O. MacAdoo; citation_firstpage=339; citation_lastpage=358; citation_publication_date=1993; citation_publisher=Academic Press; "> <meta name="citation_reference" content="citation_title=How to name a prokaryote? Etymological considerations, proposals and practical advice in prokaryote nomenclature; citation_author=H.G. Trüper; citation_journal_title=FEMS Microbiol. Rev.; citation_volume=23; citation_firstpage=231; citation_lastpage=249; citation_publication_date=1999; "> <meta name="citation_reference" content="citation_title=How to name new taxa of Archaea and Bacteria; citation_inbook_title=Bergey’s Manual of Systematics of Archaea and Bacteria; citation_author=A. Oren; citation_firstpage=1; citation_lastpage=24; citation_publication_date=2015; citation_publisher=John Wiley; "> <meta name="citation_reference" content="citation_title=Validation of the publication of new names and new combinations previously effectively published outside the IJSB. List no. 17; citation_author=Anonymous; citation_journal_title=Int. J. Syst. Bacteriol.; citation_volume=35; citation_firstpage=223; citation_lastpage=225; citation_publication_date=1985; "> <meta name="citation_reference" content="citation_title=Revised nomenclature of Campylobacter pyloridis; citation_author=B.J. Marshall; citation_author=C.S. Goodwin; citation_journal_title=Int. J. Syst. Bacteriol.; citation_volume=37; citation_firstpage=68; citation_publication_date=1987; "> <meta name="citation_reference" content="citation_title=Wanted: microbiologists with basic knowledge of Latin and Greek to join our ‘nomenclature quality control’ team; citation_author=A. Oren; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=65; citation_firstpage=3761; citation_lastpage=3762; citation_publication_date=2015; "> <meta name="citation_reference" content="citation_title=Culturing the human microbiota and culturomics; citation_author=J.C. Lagier; citation_author=et al.; citation_journal_title=Nat. Rev. Microbiol.; citation_volume=16; citation_firstpage=540; citation_lastpage=550; citation_publication_date=2018; "> <meta name="citation_reference" content="citation_title=Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life; citation_author=D.H. Parks; citation_author=et al.; citation_journal_title=Nat. Microbiol.; citation_volume=2; citation_firstpage=1533; citation_lastpage=1542; citation_publication_date=2017; "> <meta name="citation_reference" content="citation_title=A complete domain-to-species taxonomy for Bacteria and Archaea; citation_author=D.H. Parks; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=38; citation_firstpage=1079; citation_lastpage=1086; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=Naming microorganisms: the contribution of the IHU Méditerranée Infection, Marseille, France; citation_author=J.C. Lagier; citation_author=et al.; citation_journal_title=New Microbes New Infect.; citation_volume=26; citation_firstpage=S89; citation_lastpage=S95; citation_publication_date=2018; "> <meta name="citation_reference" content="citation_title=Uncultivated microbes in need of their own taxonomy; citation_author=K.T. Konstantinidis; citation_author=et al.; citation_journal_title=ISME J.; citation_volume=11; citation_firstpage=2399; citation_lastpage=2406; citation_publication_date=2017; "> <meta name="citation_reference" content="citation_title=Roadmap for naming uncultivated Archaea and Bacteria; citation_author=A.E. Murray; citation_author=et al.; citation_journal_title=Nat. Microbiol.; citation_volume=5; citation_firstpage=987; citation_lastpage=994; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=Taxonomic note: implementation of the provisional status Candidatus for incompletely described procaryotes; citation_author=R.G. Murray; citation_author=E. Stackebrandt; citation_journal_title=Int. J. Syst. Bacteriol.; citation_volume=45; citation_firstpage=186; citation_lastpage=187; citation_publication_date=1995; "> <meta name="citation_reference" content="citation_title=Lists of names of prokaryotic Candidatus taxa; citation_author=A. Oren; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=70; citation_firstpage=3956; citation_lastpage=4042; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=A genomic blueprint of the chicken gut microbiome; citation_author=R. Gilroy; citation_author=et al.; citation_journal_title=Res. Square; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=The great automatic grammatizator; citation_inbook_title=Someone Like You Alfred; citation_author=R. Dahl; citation_publication_date=1953; citation_publisher=A. Knopf; "> <meta name="citation_reference" content="citation_title=Formation of compound generic names based on personal names: a proposal for emendation of Appendix 9 of the International Code of Nomenclature of Prokaryotes; citation_author=A. Oren; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=69; citation_firstpage=594; citation_lastpage=596; citation_publication_date=2019; "> <meta name="citation_reference" content="citation_title=Naming novel prokaryotic taxa discovered in the human gut; citation_author=A. Oren; citation_journal_title=Gut; citation_volume=69; citation_firstpage=969; citation_lastpage=970; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=It’s time to retire the prokaryote; citation_author=N.R. Pace; citation_journal_title=Microbiol. Today; citation_volume=36; citation_firstpage=84; citation_publication_date=2009; "> <meta name="citation_reference" content="citation_title=Rebuttal: the modern concept of the procaryote; citation_author=N.R. Pace; citation_journal_title=J. Bacteriol.; citation_volume=191; citation_firstpage=2006; citation_lastpage=2007; citation_publication_date=2009; "> <meta name="citation_reference" content="citation_title=Problems with &#39;procaryote&#39;; citation_author=N.R. Pace; citation_journal_title=J. Bacteriol.; citation_volume=191; citation_firstpage=2008; citation_lastpage=2010; citation_publication_date=2009; "> <meta name="citation_reference" content="citation_title=Laboratory strains of Escherichia coli: model citizens or deceitful delinquents growing old disgracefully; citation_author=J.L. Hobman; citation_author=et al.; citation_journal_title=Mol. Microbiol.; citation_volume=64; citation_firstpage=881; citation_lastpage=885; citation_publication_date=2007; "> <meta name="citation_reference" content="citation_title=Minutes of the International Committee on Systematics of Prokaryotes online discussion on the proposed use of gene sequences as type for naming of prokaryotes, and outcome of vote; citation_author=I.C. Sutcliffe; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=70; citation_firstpage=4416; citation_lastpage=4417; citation_publication_date=2020; "> <meta name="citation_reference" content="citation_title=Why are so many effectively published names of prokaryotic taxa never validated; citation_author=A. Oren; citation_author=et al.; citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_volume=68; citation_firstpage=2125; citation_lastpage=2129; citation_publication_date=2018; "> <meta name="citation_abstract" content="&lt;p&gt;Latin binomials, popularised in the 18th century by the Swedish naturalist Linnaeus, have stood the test of time in providing a stable, clear, and memorable system of nomenclature across biology. However, relentless and ever-deeper exploration and analysis of the microbial world has created an urgent need for huge numbers of new names for Archaea and Bacteria. Manual creation of such names remains difficult and slow and typically relies on expert-driven nomenclatural quality control. Keen to ensure that the legacy of Linnaeus lives on in the age of microbial genomics and metagenomics, we propose an automated approach, employing combinatorial concatenation of roots from Latin and Greek to create linguistically correct names for genera and species that can be used off the shelf as needed. As proof of principle, we document over a million new names for Bacteria and Archaea. We are confident that our approach provides a road map for how to create new names for decades to come.&lt;/p&gt;"><!--

    Pagebuild admin UI
-->





<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" id="www-widgetapi-script" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/www-widgetapi.js" async=""></script><script data-dapp-detection="">!function(){let e=!1;function n(){if(!e){const n=document.createElement("meta");n.name="dapp-detected",document.head.appendChild(n),e=!0}}if(window.hasOwnProperty("ethereum")){if(window.__disableDappDetectionInsertion=!0,void 0===window.ethereum)return;n()}else{var t=window.ethereum;Object.defineProperty(window,"ethereum",{configurable:!0,enumerable:!1,set:function(e){window.__disableDappDetectionInsertion||n(),t=e},get:function(){if(!window.__disableDappDetectionInsertion){const e=arguments.callee;e&&e.caller&&e.caller.toString&&-1!==e.caller.toString().indexOf("getOwnPropertyNames")||n()}return t}})}}();</script><script type="text/javascript" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/jqjm_selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/jqjm_selected-legacy.css">
        <link rel="stylesheet" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/css">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/build-df91a08386e7f4c87488.css"><link rel="stylesheet" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/print-df91a08386e7f4c87488.css" media="print"><link rel="stylesheet" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/theme-timi.css">




<script class="optanon-category-4" rel="preconnect" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/satelliteLib-8e350dc4ceceb17a5d77c03f66d95608807a5585.js"></script>

<script class="optanon-category-4" async="" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/iframe_api"></script>

<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">






    







    
        
    












































    



<script class="optanon-category-4" type="text/javascript" async="" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/addthis_widget.js"></script><script async="" type="text/javascript" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gpt.js"></script><link rel="preload" as="script"><link rel="preconnect"><script async="" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/moatheader.js"></script><link id="plx-css-popup" type="text/css" rel="stylesheet" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/popup.css"><script type="text/javascript" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/xss.js"></script><script src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/embed.js" data-timestamp="1610964747575"></script><link href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/style.css" type="text/css" rel="stylesheet"><style></style><link rel="prefetch" as="style" href="https://c.disquscdn.com/next/embed/styles/lounge.50c6d245bed5cbae0aed0c3742f23ac0.css"><link rel="prefetch" as="script" href="https://c.disquscdn.com/next/embed/common.bundle.38ea27189bdb723eae3dabf5bc7b8c0b.js"><link rel="prefetch" as="script" href="https://c.disquscdn.com/next/embed/lounge.bundle.51e8881d4a05058fb02f6c4f09a1b290.js"><link rel="prefetch" as="script" href="https://disqus.com/next/config.js"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
<body class="pb-ui"><div id="MathJax_Message" style="display: none;"></div>
    <div class="skip">
        <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#pageBody" id="skipNavigationLink" title="Skip to Main Content" tabindex="0">Skip to Main Content</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>


    <script src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/MathJax.js" id=""></script>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->

<script type="text/javascript">
	(function () {
		var useSSL = 'https:' == document.location.protocol;

        var gads = document.createElement('script');
        gads.async = true;
        gads.type = 'text/javascript';
        gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
        document.head.appendChild(gads);

        var elsevierMoatHeader = document.createElement('link');
        elsevierMoatHeader.rel = 'preload';
        elsevierMoatHeader.as = 'script';
        elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
        document.head.appendChild(elsevierMoatHeader);

        var elsevierMoatAds = document.createElement('link');
        elsevierMoatAds.rel = 'preconnect';
        elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
        document.head.appendChild(elsevierMoatAds);

        var moatHeader = document.createElement('script');
        moatHeader.async = true; gads.type = 'text/javascript';
        moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
        document.head.appendChild(moatHeader);
	})();

</script>
<script type="text/javascript">
	//--------------------------------------------------------------------------
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Once the dom has loaded, begin the process to init the ads.
		gpt_CheckForMOATInit();
	});
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gpt_InitAds();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function GPT_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId != '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ == null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs == null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId != '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForStaging() {
		//Check to see if we are on the marlin-stag env
		//if so, set a targeting value
		if (window.location.hostname.toLowerCase().indexOf('.marlin-stag.') >= 0) {
			$('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="marlin-stag">');
			gpt_Target("Environment");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue != '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
    //--------------------------------------------------------------------------
    async function gpt_PreTearsheet(adTypeId) {
        if (typeof adTypeId === "undefined") {
            adTypeId = 0;
        }
        //Remove Elsevier Cookie Policy Notice
        if ($("#onetrust-consent-sdk").length > 0) {
            $("#onetrust-consent-sdk").remove();
        }

        //JBS Modals
        if ($('.leadinModal').length > 0) {
            $('.leadinModal').remove();
        }
        if ($('.leadinModal').length > 0) {
            $('.leadinModal').remove();
        }
        if ($('.usabilla__overlay').length > 0) {
            $('.usabilla__overlay').remove();
        }

        //Pendo
        if ($('._pendo-step-container').length > 0) {
            $('._pendo-step-container').remove();
        }
        if ($('._pendo-guide-tt_').length > 0) {
            $('._pendo-guide-tt_').remove();
        }
        if ($('._pendo-image').length > 0) {
            $('._pendo-image').remove();
        }
        if ($('._pendo-badge-image').length > 0) {
            $('._pendo-badge-image').remove();
        }

        var gptAdFile = $('#gpt-Variables').data('adfile');
        switch (gptAdFile) {
            case "JBS":
            case "Lancet":
                objLeaderboard = ".container-leaderboard";
                offset = 53;
                switch (parseInt(adTypeId)) {
                    case 3: //MPU
                    case 4: //SkyScraper
                        //BUGFIX: Show Sidebar ad for JBS
                        //skyscraper & mpu
                        if ($('.widget-verticalAd').length > 0) {
                            $('.widget-verticalAd').css('display', 'block');
                        }
                        break;
                }
                break;
        }
    }
    //--------------------------------------------------------------------------
    function gpt_Sleep(ms) {
        return new Promise(resolve => setTimeout(resolve, ms));
    }
    //--------------------------------------------------------------------------
    async function gpt_ScrollToAd(adTypeId) {
        if (typeof adTypeId === "undefined") {
            adTypeId = 0;
        }
        var adId = "";
        var offset = 0;
        var objLeaderboard = "";
        var gpt_ScrollToAd = true;

        var gptAdFile = $('#gpt-Variables').data('adfile');
        switch (gptAdFile) {
            case "PU": //PracticeUpdate
                var hideOtherAds = true;
                $('#modalPrestitial').modal('hide');
                $('.jumbotron-welcome').hide();
                objLeaderboard = ".container-leaderboard";
                offset = 53;
                switch (parseInt(adTypeId)) {
                    case 2:
                        //LeaderBoard, no Scroll Needed
                        gpt_ScrollToAd = false;
                        adId = "adSlot1";
                        break;
                    case 3:
                        //MPU
                        adId = "adSlot2";
                        break;
                    case 4:
                        //SkyScraper
                        adId = "adSlot3";
                        $('.most-read').hide();
                        break;
                    case 5:
                        //Mobile, no Scroll Needed
                        gpt_ScrollToAd = false;
                        adId = "adSlot1";
                        break;
                    case 6:
                        //Interstitial
                        gpt_ScrollToAd = false;
                        adId = "adSlot15";
                        await gpt_Sleep(1000);
                        $(objLeaderboard).hide();
                        $('#modalPrestitial').unbind();
                        $('#modalPrestitial').modal();
                        break;
                    case 7:
                        //Native Ad
                        break;
                    case 10:
                        //Master Companion
                        hideOtherAds = false;
                        gpt_ScrollToAd = false;
                        break;
                    default:
                        //Unknown Ad Type
                        gpt_ScrollToAd = false;
                        adId = "adSlot1";
                        break;
                }
                if (hideOtherAds) {
                    //Hide all other ads
                    $('.ad-container').each(function (i, obj) {
                        if (this.id !== adId) {
                            var tempParent = this.id;
                            tempParent = tempParent.replace("adSlot", "adParent");
                            $("#" + tempParent).addClass('hide');
                        }
                    });
                }
                break;
            case "JBS":
                objLeaderboard = ".js__adToHide";
                offset = 80;
                switch (parseInt(adTypeId)) {
                    case 2:
                        //Leaderboard
						gpt_ScrollToAd = false;
                        $(".widget-verticalAd").hide();
                        break;
                    case 5:
                        //Mobile
                        //Scroll to Top
                        gpt_ScrollToAd = false;
                        break;
                    case 3: //MPU
                    case 4: //SkyScraper
                        $(objLeaderboard).hide();
                        adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
                        break;
                    case 10:
                        //MasterCompanion
                        gpt_ScrollToAd = false;
                        break;
                    default:
                        //Unknown Ad Type
                        gpt_ScrollToAd = false;
                        break;
                }
                break;
            case "CellJournal":
            case "CellBucket":
                objLeaderboard = ".js__adToHide";
                offset = 260;
                switch (parseInt(adTypeId)) {
                    case 2:
                        //Leaderboard
						gpt_ScrollToAd = false;
                        $('.body-verticalAd').hide();
                        break;
                    case 5:
                        //Mobile
                        //Scroll to Top
                        gpt_ScrollToAd = false;
                        break;
                    case 3: //MPU
                    case 4: //SkyScraper
                        $(objLeaderboard).hide();
                        adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
                        break;
                    case 10:
                        //MasterCompanion
                        gpt_ScrollToAd = false;
                        break;
                    default:
                        //Unknown Ad Type
                        gpt_ScrollToAd = false;
                        break;
                }
                //Hide the second ad if found
                var tCount = 0;
                $(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
                    tCount += 1;
                    if (tCount !== 1) {
                        $(this).hide();
                    }
                });
                break;
            case "Lancet":
                objLeaderboard = ".js__adToHide";
                offset = 80;
                switch (parseInt(adTypeId)) {
                    case 2:
                        //Leaderboard
						gpt_ScrollToAd = false;
                        $('.body-verticalAd').hide();
                        break;
                    case 5:
                        //Mobile
                        //Scroll to Top
                        gpt_ScrollToAd = false;
                        break;
                    case 3: //MPU
                    case 4: //SkyScraper
                        $(objLeaderboard).hide();
                        adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
                        break;
                    case 10:
                        //MasterCompanion
                        gpt_ScrollToAd = false;
                        break;
                    default:
                        //Unknown Ad Type
                        gpt_ScrollToAd = false;
                        break;
                }
                break;
        }

        //Scroll to Top
        var gpt_container = $("html,body");
        gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

        if (gpt_ScrollToAd) {
            //If Adexists, scroll to it
            var gpt_scrollTo = $('#' + adId);
            if (gpt_scrollTo.length > 0) {
                //Scroll to the Ad
                var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
                if (gpt_topPosition > 0) {
                    gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
                    if (gpt_topPosition - offset > 0 && offset > 0) {
                        gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
                    }
                }
            }
        }
    }
    //--------------------------------------------------------------------------
	function gpt_InitAds() {
		googletag.cmd.push(function () {

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			var gptLeaderboardSizeMapping = googletag.sizeMapping().
				addSize([992, 0], [[728, 90]]). 						// Desktop
				addSize([768, 0], [[728, 90]]). 								// Tablet Landscape
				addSize([0, 0], [[320, 50], [300, 50]]). 						// Phone & Tablet Portrait
				build();

			var gptSideBarSizeMapping = googletag.sizeMapping().
                addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]). 	// SkyScraper - Desktop (Require min 630 vertical)
				addSize([1140, 445], [[300, 250]]).			 				// BoomBox Fall Back - Desktop (Require min 280 vertical)
				addSize([0, 0], []). 										// Phone
				build();

			var gptBoomBoxSizeMapping = googletag.sizeMapping().
				addSize([768, 0], [[300, 250],[336, 280]]). 					// Desktop & Tablet Landscape
				addSize([0, 0], []). 											// Phone
				build();

            var gptFluidMapping = googletag.sizeMapping().
                addSize([0, 0], ['fluid']).
                build();

			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = GPT_CheckForNULL($('#gptPage').val());
			var gptMasterSite = GPT_CheckForNULL($('#gptSite').val());

			//If there is no master site, check for a defaultsite
			if (gptMasterSite == null) {
				gptMasterSite = GPT_CheckForNULL($('#gptDefaultSite').val());
			}

			//If there is no master Page, check for a Page using the URL
			if (gptMasterPage == null) {
				gptMasterPage = GPT_GetPageByURL();
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var adId;
				var gptPOS;
				var gptSite = null;
				var tempSlot = null;

				//Check if the ad slot has it's own site defined
				if (!typeof $(this).data('site') == 'undefined') {
					gptSite = GPT_CheckForNULL($(this).data('site'));
				}
				else {
					//Use the Master Site
					gptSite = gptMasterSite;
				}

				if (gptSite != null) {
					adId = 'gpt-ad-' + index + 1;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS == -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					if (!typeof $(this).data('page') == 'undefined') {
						gptPage = GPT_CheckForNULL($(this).data('page'));
					}
					else {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage != null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}

					//Determine sizemapping based on data-adsize
					switch ($(this).data('adsize')) {
						case 'cell-leaderboard':
						case 'leaderboard':
							adSizeMapping = gptLeaderboardSizeMapping;
							tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
							break;
						case 'bottom':
						case 'boombox':
							adSizeMapping = gptBoomBoxSizeMapping;
                            tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
							break;
						case 'sidebar':
						case 'skyscraper':
							adSizeMapping = gptSideBarSizeMapping;
							tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
							break;
                        case 'native':
                        case 'fluid':
                            adSizeMapping = gptFluidMapping;
                            tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
                            break;
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping != '') {
					$(this).addClass('gpt-ad-defined');
					$(this).attr('id', adId);
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);
				}
				else {
					$(this).remove();
				}
			});
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			//Logged in or IP Access
			gpt_TargetAccessType();

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Staging
			gpt_CheckForStaging();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting()

			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.enableServices();

			//Add Listener for Leaderboard
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					$('#' + event.slot.getSlotElementId()).closest(".GPT-ad-placeholder").css("display", "block");
				}
			});

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

		});
	}
</script>
<div id="gpt-Variables" data-adfile="CellJournal" data-version="01152021">
	<input type="hidden" id="gptSite" value="timi">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<!-- End: GPT -->

        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d" style="display: flex;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd " style="display: none;"><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad gpt-ad-defined" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1" id="gpt-ad-01"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow" style="display: block;"><a href="https://www.cell.com/" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight">A Cell Press journal</p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6">
    

        


    

            

            
                <svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13	C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2	l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92 	c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8	c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="https://www.cell.com/trends/microbiology/home" title="Trends in Microbiology"><img alt="Trends in Microbiology" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/TrendsinMicrobiology_white.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row menu--desk">
    

        


    

            

            

            <div data-widget-id="ae520d74-989f-4741-a72a-da74d1984826" data-widget-name="UX3MenuWidget" class="main-nav menu-drawer"><div class="menu-drawer__ctrl"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#main-menu" data-target="main-menu" data-toggle="nav" title="Explore Cell.com" aria-controls="main-menu" aria-expanded="false"><span class="menu-drawer--open"><i aria-hidden="true" class="icon-gizmo-hamburger"></i></span><span class="menu-drawer--close"><i aria-hidden="true" class="icon-close_thin"><span>Close</span></i></span></a></div><nav role="navigation" id="main-menu" data-ctrl-res="screen-sm" class="menu-drawer__nav drawer__nav gutterless" style="top: 115.2px;"><span class="menu-drawer--holder hidden-md hidden-lg"></span><ul id="menubar" role="menubar" class="menubar rlist--inline"><li role="menuitem" aria-haspopup="true" aria-label="This Journal" id="menu-item-main-menu-0" class="dropdown menu-parent is--current"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="This Journal" id="main-menu-0" data-toggle="dropdown" class="dropdown__toggle"><span>This Journal</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-0" role="menu" class="rlist dropdown__menu"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Online Now" class="menu-item"><a href="https://www.cell.com/trends/microbiology/newarticles" title="Online Now"><span>Online Now</span></a></li><li role="menuitem" tabindex="-1" aria-label="Current Issue" class="menu-item"><a href="https://www.cell.com/trends/microbiology/current" title="Current Issue"><span>Current Issue</span></a></li><li role="menuitem" tabindex="-1" aria-label="About the Journal" class="menu-item"><a href="https://www.cell.com/trends/microbiology/about" title="About the Journal"><span>About the Journal</span></a></li><li role="menuitem" tabindex="-1" aria-label="For Authors" class="menu-item"><a href="https://www.cell.com/trends/microbiology/author-resources" title="For Authors"><span>For Authors</span></a></li><li role="menuitem" tabindex="-1" aria-label="Archive" class="menu-item"><a href="https://www.cell.com/trends/microbiology/archive" title="Archive"><span>Archive</span></a></li><li role="menuitem" tabindex="-1" aria-label="Alerts" class="menu-item"><a href="https://www.cell.com/action/showPreferences?menuTab=Alerts&amp;code=cell-site+" title="Alerts"><span>Alerts</span></a></li><li role="menuitem" tabindex="-1" aria-label="Submit" class="menu-item"><a href="https://www.editorialmanager.com/timi/default.aspx" title="Submit"><span>Submit</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Research Journals" id="menu-item-main-menu-1" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Research Journals" id="main-menu-1" data-toggle="dropdown" class="dropdown__toggle"><span>Research Journals</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-1" role="menu" class="rlist dropdown__menu level-2"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell" class="menu-item"><a href="https://www.cell.com/cell/home" title="Cell"><span>Cell</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cancer Cell" class="menu-item"><a href="https://www.cell.com/cancer-cell/home" title="Cancer Cell"><span>Cancer Cell</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Chemical Biology" class="menu-item"><a href="https://www.cell.com/cell-chemical-biology/home" title="Cell Chemical Biology"><span>Cell Chemical Biology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Genomics" class="menu-item"><a href="https://www.cell.com/cell-genomics/home" title="Cell Genomics"><span>Cell Genomics</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Host &amp; Microbe" class="menu-item"><a href="https://www.cell.com/cell-host-microbe/home" title="Cell Host &amp; Microbe"><span>Cell Host &amp; Microbe</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Metabolism" class="menu-item"><a href="https://www.cell.com/cell-metabolism/home" title="Cell Metabolism"><span>Cell Metabolism</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Reports" class="menu-item"><a href="https://www.cell.com/cell-reports/home" title="Cell Reports"><span>Cell Reports</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Reports Medicine" class="menu-item"><a href="https://www.cell.com/cell-reports-medicine/home" title="Cell Reports Medicine"><span>Cell Reports Medicine</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Reports Methods" class="menu-item"><a href="https://www.cell.com/cell-reports-methods/home" title="Cell Reports Methods"><span>Cell Reports Methods</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Reports Physical Science" class="menu-item"><a href="https://www.cell.com/cell-reports-physical-science/home" title="Cell Reports Physical Science"><span>Cell Reports Physical Science</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Stem Cell" class="menu-item"><a href="https://www.cell.com/cell-stem-cell/home" title="Cell Stem Cell"><span>Cell Stem Cell</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Systems" class="menu-item"><a href="https://www.cell.com/cell-systems/home" title="Cell Systems"><span>Cell Systems</span></a></li><li role="menuitem" tabindex="-1" aria-label="Chem" class="menu-item"><a href="https://www.cell.com/chem/home" title="Chem"><span>Chem</span></a></li><li role="menuitem" tabindex="-1" aria-label="Chem Catalysis" class="menu-item"><a href="https://www.cell.com/chem-catalysis/home" title="Chem Catalysis"><span>Chem Catalysis</span></a></li><li role="menuitem" tabindex="-1" aria-label="Current Biology" class="menu-item"><a href="https://www.cell.com/current-biology/home" title="Current Biology"><span>Current Biology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Developmental Cell" class="menu-item"><a href="https://www.cell.com/developmental-cell/home" title="Developmental Cell"><span>Developmental Cell</span></a></li><li role="menuitem" tabindex="-1" aria-label="Heliyon" class="menu-item"><a href="https://www.cell.com/heliyon/home" title="Heliyon"><span>Heliyon</span></a></li><li role="menuitem" tabindex="-1" aria-label="Immunity" class="menu-item"><a href="https://www.cell.com/immunity/home" title="Immunity"><span>Immunity</span></a></li><li role="menuitem" tabindex="-1" aria-label="iScience" class="menu-item"><a href="https://www.cell.com/iscience/home" title="iScience"><span>iScience</span></a></li><li role="menuitem" tabindex="-1" aria-label="Joule" class="menu-item"><a href="https://www.cell.com/joule/home" title="Joule"><span>Joule</span></a></li><li role="menuitem" tabindex="-1" aria-label="Matter" class="menu-item"><a href="https://www.cell.com/matter/home" title="Matter"><span>Matter</span></a></li><li role="menuitem" tabindex="-1" aria-label="Med" class="menu-item"><a href="https://www.cell.com/med/home" title="Med"><span>Med</span></a></li><li role="menuitem" tabindex="-1" aria-label="Molecular Cell" class="menu-item"><a href="https://www.cell.com/molecular-cell/home" title="Molecular Cell"><span>Molecular Cell</span></a></li><li role="menuitem" tabindex="-1" aria-label="Neuron" class="menu-item"><a href="https://www.cell.com/neuron/home" title="Neuron"><span>Neuron</span></a></li><li role="menuitem" tabindex="-1" aria-label="One Earth" class="menu-item"><a href="https://www.cell.com/one-earth/home" title="One Earth"><span>One Earth</span></a></li><li role="menuitem" tabindex="-1" aria-label="Patterns" class="menu-item"><a href="https://www.cell.com/patterns/home" title="Patterns"><span>Patterns</span></a></li><li role="menuitem" tabindex="-1" aria-label="STAR Protocols" class="menu-item"><a href="https://www.cell.com/star-protocols/home" title="STAR Protocols"><span>STAR Protocols</span></a></li><li role="menuitem" tabindex="-1" aria-label="Structure" class="menu-item"><a href="https://www.cell.com/structure/home" title="Structure"><span>Structure</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Trends Reviews Journals" id="menu-item-main-menu-2" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Trends Reviews Journals" id="main-menu-2" data-toggle="dropdown" class="dropdown__toggle"><span>Trends Reviews Journals</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-2" role="menu" class="rlist dropdown__menu level-3"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="About Trends Reviews Journals" class="menu-item"><a href="https://www.cell.com/trends" title="About Trends Reviews Journals"><span>About Trends Reviews Journals</span></a></li><li role="menuitem" tabindex="-1" aria-label="Biochemical Sciences" class="menu-item"><a href="https://www.cell.com/trends/biochemical-sciences/home" title="Biochemical Sciences"><span>Biochemical Sciences</span></a></li><li role="menuitem" tabindex="-1" aria-label="Biotechnology" class="menu-item"><a href="https://www.cell.com/trends/biotechnology/home" title="Biotechnology"><span>Biotechnology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cancer" class="menu-item"><a href="https://www.cell.com/trends/cancer/home" title="Cancer"><span>Cancer</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Biology" class="menu-item"><a href="https://www.cell.com/trends/cell-biology/home" title="Cell Biology"><span>Cell Biology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Chemistry" class="menu-item"><a href="https://www.cell.com/trends/chemistry/home" title="Chemistry"><span>Chemistry</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cognitive Sciences" class="menu-item"><a href="https://www.cell.com/trends/cognitive-sciences/home" title="Cognitive Sciences"><span>Cognitive Sciences</span></a></li><li role="menuitem" tabindex="-1" aria-label="Ecology &amp; Evolution" class="menu-item"><a href="https://www.cell.com/trends/ecology-evolution/home" title="Ecology &amp; Evolution"><span>Ecology &amp; Evolution</span></a></li><li role="menuitem" tabindex="-1" aria-label="Endocrinology &amp; Metabolism" class="menu-item"><a href="https://www.cell.com/trends/endocrinology-metabolism/home" title="Endocrinology &amp; Metabolism"><span>Endocrinology &amp; Metabolism</span></a></li><li role="menuitem" tabindex="-1" aria-label="Genetics" class="menu-item"><a href="https://www.cell.com/trends/genetics/home" title="Genetics"><span>Genetics</span></a></li><li role="menuitem" tabindex="-1" aria-label="Immunology" class="menu-item"><a href="https://www.cell.com/trends/immunology/home" title="Immunology"><span>Immunology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Microbiology" class="menu-item"><a href="https://www.cell.com/trends/microbiology/home" title="Microbiology"><span>Microbiology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Molecular Medicine" class="menu-item"><a href="https://www.cell.com/trends/molecular-medicine/home" title="Molecular Medicine"><span>Molecular Medicine</span></a></li><li role="menuitem" tabindex="-1" aria-label="Neurosciences" class="menu-item"><a href="https://www.cell.com/trends/neurosciences/home" title="Neurosciences"><span>Neurosciences</span></a></li><li role="menuitem" tabindex="-1" aria-label="Parasitology" class="menu-item"><a href="https://www.cell.com/trends/parasitology/home" title="Parasitology"><span>Parasitology</span></a></li><li role="menuitem" tabindex="-1" aria-label="Pharmacological Sciences" class="menu-item"><a href="https://www.cell.com/trends/pharmacological-sciences/home" title="Pharmacological Sciences"><span>Pharmacological Sciences</span></a></li><li role="menuitem" tabindex="-1" aria-label="Plant Science" class="menu-item"><a href="https://www.cell.com/trends/plant-science/home" title="Plant Science"><span>Plant Science</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Partner Journals" id="menu-item-main-menu-3" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Partner Journals" id="main-menu-3" data-toggle="dropdown" class="dropdown__toggle"><span>Partner Journals</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-3" role="menu" class="rlist dropdown__menu level-4"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="AJHG" class="menu-item"><a href="https://www.cell.com/ajhg/home" title="AJHG"><span>AJHG</span></a></li><li role="menuitem" tabindex="-1" aria-label="Biophysical Journal" class="menu-item"><a href="https://www.cell.com/biophysj/home" title="Biophysical Journal"><span>Biophysical Journal</span></a></li><li role="menuitem" tabindex="-1" aria-label="EBioMedicine" class="menu-item"><a href="https://www.ebiomedicine.com/" title="EBioMedicine"><span>EBioMedicine</span></a></li><li role="menuitem" tabindex="-1" aria-label="HGG Advances" class="menu-item"><a href="https://www.cell.com/hgg-advances/home" title="HGG Advances"><span>HGG Advances</span></a></li><li role="menuitem" tabindex="-1" aria-label="Molecular Plant" class="menu-item"><a href="https://www.cell.com/molecular-plant/home" title="Molecular Plant"><span>Molecular Plant</span></a></li><li role="menuitem" tabindex="-1" aria-label="Molecular Therapy Family" class="menu-item"><a href="https://www.cell.com/molecular-therapy-family/home" title="Molecular Therapy Family"><span>Molecular Therapy Family</span></a></li><li role="menuitem" tabindex="-1" aria-label="Plant Communications" class="menu-item"><a href="https://www.cell.com/plant-communications/home" title="Plant Communications"><span>Plant Communications</span></a></li><li role="menuitem" tabindex="-1" aria-label="Stem Cell Reports" class="menu-item"><a href="https://www.cell.com/stem-cell-reports/home" title="Stem Cell Reports"><span>Stem Cell Reports</span></a></li><li role="menuitem" tabindex="-1" aria-label="The Innovation" class="menu-item"><a href="https://www.cell.com/the-innovation/home" title="The Innovation"><span>The Innovation</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Collections" id="menu-item-main-menu-4" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Collections" id="main-menu-4" data-toggle="dropdown" class="dropdown__toggle"><span>Collections</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-4" role="menu" class="rlist dropdown__menu level-5"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Best of Cell Press" class="menu-item"><a href="https://www.cell.com/bestof" title="Best of Cell Press"><span>Best of Cell Press</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Press Reviews" class="menu-item"><a href="https://www.cell.com/reviews" title="Cell Press Reviews"><span>Cell Press Reviews</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Press Selections" class="menu-item"><a href="https://www.cell.com/selections" title="Cell Press Selections"><span>Cell Press Selections</span></a></li><li role="menuitem" tabindex="-1" aria-label="Consortia Hub" class="menu-item"><a href="https://www.cell.com/consortia" title="Consortia Hub"><span>Consortia Hub</span></a></li><li role="menuitem" tabindex="-1" aria-label="Nucleus Collections" class="menu-item"><a href="https://www.cell.com/nucleus" title="Nucleus Collections"><span>Nucleus Collections</span></a></li><li role="menuitem" tabindex="-1" aria-label="SnapShot Archive" class="menu-item"><a href="https://www.cell.com/snapshots" title="SnapShot Archive"><span>SnapShot Archive</span></a></li><li role="menuitem" tabindex="-1" aria-label="Trends Limited Editions" class="menu-item"><a href="https://www.cell.com/trends/limited-edition" title="Trends Limited Editions"><span>Trends Limited Editions</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Beyond the Journal" id="menu-item-main-menu-5" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Beyond the Journal" id="main-menu-5" data-toggle="dropdown" class="dropdown__toggle"><span>Beyond the Journal</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-5" role="menu" class="rlist dropdown__menu level-6"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Career Network" class="menu-item"><a href="http://careers.cell.com/" title="Cell Career Network"><span>Cell Career Network</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Mentor" class="menu-item"><a href="https://www.cell.com/cell-mentor" title="Cell Mentor"><span>Cell Mentor</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Symposia" class="menu-item"><a href="https://www.cell.com/symposia" title="Cell Symposia"><span>Cell Symposia</span></a></li><li role="menuitem" tabindex="-1" aria-label="CrossTalk" class="menu-item"><a href="http://crosstalk.cell.com/" title="CrossTalk"><span>CrossTalk</span></a></li><li role="menuitem" tabindex="-1" aria-label="LabLinks" class="menu-item"><a href="https://www.cell.com/lablinks" title="LabLinks"><span>LabLinks</span></a></li><li role="menuitem" tabindex="-1" aria-label="Webinars" class="menu-item"><a href="https://www.cell.com/webinars" title="Webinars"><span>Webinars</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Evolving the Article" id="menu-item-main-menu-6" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Evolving the Article" id="main-menu-6" data-toggle="dropdown" class="dropdown__toggle"><span>Evolving the Article</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-6" role="menu" class="rlist dropdown__menu level-7"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Figure 360" class="menu-item"><a href="https://www.cell.com/figure360" title="Figure 360"><span>Figure 360</span></a></li><li role="menuitem" tabindex="-1" aria-label="Sneak Peek" class="menu-item"><a href="https://www.cell.com/sneakpeek" title="Sneak Peek"><span>Sneak Peek</span></a></li><li role="menuitem" tabindex="-1" aria-label="STAR Methods" class="menu-item"><a href="https://www.cell.com/star-methods" title="STAR Methods"><span>STAR Methods</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Science in Society" id="menu-item-main-menu-7" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Science in Society" id="main-menu-7" data-toggle="dropdown" class="dropdown__toggle"><span>Science in Society</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-7" role="menu" class="rlist dropdown__menu level-8"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Picture Show" class="menu-item"><a href="https://www.cell.com/pictureshow" title="Cell Picture Show"><span>Cell Picture Show</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Press Podcast" class="menu-item"><a href="https://www.cell.com/podcast" title="Cell Press Podcast"><span>Cell Press Podcast</span></a></li><li role="menuitem" tabindex="-1" aria-label="Cell Press Videos" class="menu-item"><a href="https://www.cell.com/video" title="Cell Press Videos"><span>Cell Press Videos</span></a></li><li role="menuitem" tabindex="-1" aria-label="Coloring and Comics" class="menu-item"><a href="https://www.cell.com/comics" title="Coloring and Comics"><span>Coloring and Comics</span></a></li><li role="menuitem" tabindex="-1" aria-label="Research Arc" class="menu-item"><a href="https://www.cell.com/research-arc" title="Research Arc"><span>Research Arc</span></a></li></ul></li><li role="menuitem" aria-haspopup="true" aria-label="Connect" id="menu-item-main-menu-8" class="dropdown menu-parent"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Connect" id="main-menu-8" data-toggle="dropdown" class="dropdown__toggle"><span>Connect</span><i aria-hidden="true" class="icon-arrow_r pull-right"></i></a><ul id="menu-items-8" role="menu" class="rlist dropdown__menu level-9"><li role="menuitem" aria-haspopup="true" class="menu-drawer__back hidden-md hidden-lg"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="menu-drawer__back__ctrl"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back</span></a></li><li role="menuitem" tabindex="-1" aria-label="About Cell Press" class="menu-item"><a href="https://www.cell.com/about" title="About Cell Press"><span>About Cell Press</span></a></li><li role="menuitem" tabindex="-1" aria-label="Careers" class="menu-item"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" title="Careers"><span>Careers</span></a></li><li role="menuitem" tabindex="-1" aria-label="Contact" class="menu-item"><a href="https://www.cell.com/contact" title="Contact"><span>Contact</span></a></li><li role="menuitem" tabindex="-1" aria-label="Help &amp;amp; Support" class="menu-item"><a href="https://service.elsevier.com/app/overview/cell-press/" title="Help &amp;amp; Support"><span>Help &amp; Support</span></a></li><li role="menuitem" tabindex="-1" aria-label="Newsroom" class="menu-item"><a href="https://www.cell.com/newsroom" title="Newsroom"><span>Newsroom</span></a></li><li role="menuitem" tabindex="-1" aria-label="Publication Alerts" class="menu-item"><a href="https://www.cell.com/action/showPreferences?menuTab=Alerts&amp;code=cell-site" title="Publication Alerts"><span>Publication Alerts</span></a></li></ul></li><li role="menuitem" aria-label="Cell Press Home" id="menu-item-main-menu-9" class="menu-item"><a href="https://www.cell.com/home" title="Cell Press Home"><span>Cell Press Home</span></a></li><li role="menuitem" aria-label="Home" id="menu-item-main-menu-reg" class="menu-item hidden-lg"><a href="https://www.cell.com/action/registration?journalCode=timi&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Ftrends%2Fmicrobiology%2Ffulltext%2FS0966-842X%2820%2930271-7%3Frss%3Dyes&amp;directReg=true" title="Register a username and password for free">Register</a></li><li role="menuitem" aria-label="Home" id="menu-item-main-menu-sub" class="menu-item hidden-lg"><a href="https://www.cell.com/trends/microbiology/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li role="menuitem" aria-label="Home" id="menu-item-main-menu-cla" class="menu-item hidden-lg"><a href="https://www.cell.com/trends/microbiology/activate" title="Claim your online subscription">Claim</a></li></ul></nav></div><div class="header__dropzone">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="AtyponUX3QuickSearchWidget" class="quick-search atypon-quick-search"><div class="dropBlock"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Search" data-db-target-for="quickSearch" aria-expanded="false" aria-controls="quick-search-holder" class="search-drawer"><i aria-hidden="true" class="icon-search"></i></a><div data-db-target-of="quickSearch" id="quick-search-holder" class="dropBlock__holder quick-search__holder"><div class="tab row clearfix quick-search__wrapper"><form action="https://www.cell.com/action/doSearch?journalCode=timi" name="quickSearchBar" method="get" title="Quick Search"><div class="clearfix"><div class="col-sm-5 col-md-6 col-lg-7"><div class="input-group quick-search__searchbar"><input type="text" name="text1" placeholder="Search for..." autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></div><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button></div><div class="col-sm-4 col-md-4 col-lg-3"><select name="field1" aria-label="Select a search area" class="quick-search__dropdown"><option value="AllField">All Content</option><option value="Title">Article Title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article Title, Abstract, Keywords</option></select><div class="quick-search__scope-options clearfix"><input type="hidden" name="journalCode" value="timi"><label title="This Journal"><span>This Journal</span><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This Journal" value="timi" checked="checked" aria-label="Search within current journal"></label><label title="Full Site"><span>Full Site</span><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full Site" aria-label="Search within full site"></label></div></div><div class="col-sm-3 col-md-2 col-lg-2"><div class="quick-search__results-count visible-xs visible-sm"></div><div class="quick-search__advanced-region"><span><a href="https://www.cell.com/trends/microbiology/search?SeriesKey=timi&amp;ISSN=0966-842X&amp;journalCode=timi" title="Advanced search" class="quick-search__advanced-link">Advanced Search</a></span><span class="quick-search__save-search visible-xs visible-sm"><a href="https://www.cell.com/action/doSaveSearch" role="button" data-toggle="modal" data-target="#saveSearchDialog" aria-label="Click here to save your search" title="Save search">Save search</a></span></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="quick-search__alert--close"> Ok</a></div></div></form><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close search bar" class="quick-search__ctrl quick-search__close hidden-xs hidden-sm"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div></div></div>

            

        
</div>

            

        

    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__logIn"><a href="https://www.cell.com/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Ftrends%2Fmicrobiology%2Ffulltext%2FS0966-842X%2820%2930271-7%3Frss%3Dyes&amp;pii=S0966-842X%2820%2930271-7" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="https://www.cell.com/action/registration?journalCode=timi&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Ftrends%2Fmicrobiology%2Ffulltext%2FS0966-842X%2820%2930271-7%3Frss%3Dyes&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="https://www.cell.com/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Ftrends%2Fmicrobiology%2Ffulltext%2FS0966-842X%2820%2930271-7%3Frss%3Dyes&amp;pii=S0966-842X%2820%2930271-7" title="Log in to Cell.com" class="show-login">Log in</a></div></li><li class="profile-links__subscribe"><a href="https://www.cell.com/trends/microbiology/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="https://www.cell.com/trends/microbiology/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="520cbcfe-653d-4915-9646-9031eab22389" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.tim.2020.10.009">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content" style="padding-top: 115.2px;">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S0966-842X(20)30271-7"><div id="articleHeader" class="article__header low-height"><div class="article-header article-header__decrease-padding"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img aria-hidden="true" class="what-image b-lazy b-loaded ux-lazyloaded" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gr1.jpg"></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Opinion</span><span class="article-header__sep">|</span><a href="https://www.cell.com/timi/newarticles" class="article-header__vol faded">Online Now</a><span></span><span class="article-header__date faded"></span></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper" style="top: 0px;"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")-->The Next Million Names for Archaea and Bacteria</span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><a href="https://www.cell.com/action/showPdf?pii=S0966-842X%2820%2930271-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF [1 MB]</span><span class="alternative">Download PDF [1 MB]</span></a></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/trends/microbiology/ppt/S0966-842X(20)30271-7.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.tim.2020.10.009" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.tim.2020.10.009&amp;pii=S0966-842X%2820%2930271-7" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.tim.2020.10.009" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item">
    

        


    

            

            

            <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/addthis_widget(1).js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span>Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email"></i><span>Email</span></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li><a role="link" class="addthis_button_sinaweibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://reprints.elsevier.com/index.php?option=com_oscommerce&amp;amp;osMod=raq&amp;amp;Itemid=7&amp;amp;publicationName=Trends%20in%20Microbiology&amp;amp;title=The%20Next%20Million%20Names%20for%20Archaea%20and%20Bacteria&amp;amp;creator=Mark+J.+Pallen+and+colleagues&amp;amp;volume=0&amp;amp;number=0&amp;amp;startingPage=&amp;amp;endingPage=" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0966842X20302717&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">The Next Million Names for Archaea and Bacteria</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" data-db-target-for="au0005-authorcard" aria-expanded="false" aria-controls="au0005" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mark J. Pallen">Mark J. Pallen<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au0005-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mark J. Pallen</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Correspondence:</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="mailto:mark.pallen@quadram.ac.uk" title="Send email to Mark J. Pallen"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Quadram Institute Bioscience, Norwich Research Park, Norwich, UK<p></p>School of Veterinary Medicine, University of Surrey, Guildford, Surrey, UK<p></p>University of East Anglia, Norwich Research Park, Norwich, UK</div></div><a href="https://www.cell.com/action/doSearch?ContribStored=Pallen%2C+Mark%20J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Mark J. Pallen" class="loa__item__name  loa__item__email" href="mailto:mark.pallen@quadram.ac.uk"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" data-db-target-for="au0010-authorcard" aria-expanded="false" aria-controls="au0010" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Andrea Telatin">Andrea Telatin</a><div data-db-target-of="au0010-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Andrea Telatin</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Quadram Institute Bioscience, Norwich Research Park, Norwich, UK</div></div><a href="https://www.cell.com/action/doSearch?ContribStored=Telatin%2C+Andrea" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" data-db-target-for="au0015-authorcard" aria-expanded="false" aria-controls="au0015" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Aharon Oren">Aharon Oren</a><div data-db-target-of="au0015-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Aharon Oren</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The Institute of Life Sciences, Edmond J. Safra Campus, The Hebrew University of Jerusalem, Jerusalem, Israel</div></div><a href="https://www.cell.com/action/doSearch?ContribStored=Oren%2C+Aharon" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li> </ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">December 04, 2020</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.tim.2020.10.009">https://doi.org/10.1016/j.tim.2020.10.009</a></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script type="text/javascript" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/widget-popup.js"></script><div class="plum-jbs-theme"><div class="PlumX-Popup"><div class="ppp-container ppp-small ppp-badge"><a target="_blank" class="plx-wrapping-print-link" href="https://plu.mx/plum/a/?doi=10.1016/j.tim.2020.10.009&amp;theme=plum-jbs-theme&amp;hideUsage=true" title="PlumX Metrics Detail Page"><div class="plx-print"><svg viewBox="0 0 100 100" width="100%" height="100%" aria-labelledby="widget-plumprint-1"><title id="widget-plumprint-1">Plum Print visual indicator of research metrics</title><path fill="#6e1a62" stroke="#6e1a62" stroke-width="1" d="M 32.88410159590142,65.84746899224614 C 45.706279705045,46.39714088412485 53.44307527039509,52.151472212786814 45.08508773426605,66.85854138189372 L 54.91491226573395,66.85854138189372 C 46.55692472960491,52.151472212786814 52.75775999522832,47.68596460512846 63.924475253123596,57.96956599135724 L 65.94953047442182,51.73708687489691 C 50.87091882415881,53.493064614593585 48.651416263702714,46.662138123559565 61.88240715909315,39.21976840364822 L 56.58074376063895,35.36788447550181 C 53.59123058093537,50.25112330547885 46.40876941906463,50.25112330547885 43.419256239361054,35.36788447550181 L 38.11759284090686,39.21976840364822 C 51.348583736297286,46.662138123559565 48.6440147666877,55.43855971992513 26.838038757284664,47.23960092910828 Z"></path><circle fill="#13b7ea" stroke="#6e1a62" stroke-width="1" r="11.94251450533924" cx="23.346385432682624" cy="58.66028434872173"></circle><circle fill="none" stroke="#ffffff" stroke-width="1.5" r="10.44251450533924" cx="23.346385432682624" cy="58.66028434872173"></circle><circle fill="#f3c316" stroke="#6e1a62" stroke-width="1" r="6" cx="50" cy="70.3"></circle><circle fill="none" stroke="#ffffff" stroke-width="1.5" r="4.5" cx="50" cy="70.3"></circle><circle fill="#6e1a62" stroke="#6e1a62" stroke-width="1" r="4" cx="67.11901729331277" cy="55.56230589874905"></circle><circle fill="#6e1a62" stroke="#6e1a62" stroke-width="1" r="4" cx="60.58013454126452" cy="35.43769410125095"></circle><circle fill="#6e1a62" stroke="#6e1a62" stroke-width="1" r="4" cx="39.41986545873549" cy="35.43769410125095"></circle></svg></div>PlumX Metrics</a><div class="ppp-pop ppp-pop-left"><div class="ppp-branding"><img alt="plumX logo" class="plx-logo" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/plumx-inverse-logo.png"></div><ul><li class="plx-mention"><div class="ppp-title">Mentions</div><ul><li class="plx-mention"><span class="ppp-label">News Mentions: </span><span class="ppp-count">2</span></li></ul></li><li class="plx-socialMedia"><div class="ppp-title">Social Media</div><ul><li class="plx-socialMedia"><span class="ppp-label">Shares, Likes &amp; Comments: </span><span class="ppp-count">6</span></li><li class="plx-socialMedia"><span class="ppp-label">Tweets: </span><span class="ppp-count">117</span></li></ul></li></ul><a target="_blank" href="https://plu.mx/plum/a/?doi=10.1016/j.tim.2020.10.009&amp;theme=plum-jbs-theme&amp;hideUsage=true" title="PlumX Metrics Detail Page">see details</a></div></div></div></div></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30296-1" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Unappreciated Roles for K+ Channels in Bacterial Physiology</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30270-5" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">Joining European Scientific Forces to Face Pandemics</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1 js--sticky" style="height: 272px;"><ul class="rlist left-side-nav__upper"><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0005"><span>Highlights</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0010"><span>Keywords</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0015"><span>The Legacy of Linnaeus</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0020"><span>Naming the Unnamed Millions</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0025"><span>Automating the Creation of Names</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0065"><span>Concluding Remarks and Future Perspectives</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0070"><span>Acknowledgments</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#supplementaryMaterial"><span>Supplemental Information</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0085"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#secst0090"><span>Glossary</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#articleInformation"><span>Article Info</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#figures"><span>Figures</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#tables"><span>Tables</span></a></li><li role="menuitem"><a class="w-slide__hide" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#comments-heading"><span>Comments</span></a></li></ul><ul class="rlist left-side-nav__bottom hidden"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-no  widget-compact-all" id="69206a6d-45ef-4abc-b439-22ed5ed753b6">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-no  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes';

if(URL_var.indexOf("S0165-6147(20)30251-0") > -1
|| URL_var.indexOf("S1471-4914(20)30191-X") > -1 
|| URL_var.indexOf("S1931-3128(20)30522-9") > -1 
|| URL_var.indexOf("S2405-4712(20)30333-1") > -1 
|| URL_var.indexOf("S2666-3791(20)30127-0") > -1 
|| URL_var.indexOf("S2405-8033(20)30277-6") > -1 
    {
document.write('<input type="hidden" id="gptPage" value="rarc">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel" style="display: none;">
                <img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secst0005" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="l0005"><li class="ce-list--remove-bullets__list-item" id="li0005"><div class="ce-list--remove-bullets__list-item__text">Microbiology has entered a golden era of discovery, with exponential growth in the identification of new species, genera, and high-level taxa through culturomics, genomics, and metagenomics.</div></li><li class="ce-list--remove-bullets__list-item" id="li0010"><div class="ce-list--remove-bullets__list-item__text">This creates an urgent unmet need for new taxonomic names for Archaea and Bacteria.</div></li><li class="ce-list--remove-bullets__list-item" id="li0015"><div class="ce-list--remove-bullets__list-item__text">Currently, creation of well-formed names relies on time-consuming nomenclatorial quality control by a dwindling pool of experts conversant with classical languages and the International Code of Nomenclature of Prokaryotes.</div></li><li class="ce-list--remove-bullets__list-item" id="li0020"><div class="ce-list--remove-bullets__list-item__text">These problems are compounded by the custom of creating names on an as-needed, just-in-time-fashion.</div></li><li class="ce-list--remove-bullets__list-item" id="li0025"><div class="ce-list--remove-bullets__list-item__text">Here, we outline a novel approach with three features: creation of names en masse before they are tied to taxa; combinatorial concatenation of roots from Latin and Greek, drawing on stocks of roots with relevant meanings; computerised automation of the creation of new names.</div></li></ul></div></section><section><div class="section-paragraph"><div class="section-paragraph">Latin binomials, popularised in the 18th century by the Swedish naturalist Linnaeus, have stood the test of time in providing a stable, clear, and memorable system of nomenclature across biology. However, relentless and ever-deeper exploration and analysis of the microbial world has created an urgent need for huge numbers of new names for Archaea and Bacteria. Manual creation of such names remains difficult and slow and typically relies on expert-driven nomenclatural quality control. Keen to ensure that the legacy of Linnaeus lives on in the age of microbial genomics and metagenomics, we propose an automated approach, employing combinatorial concatenation of roots from Latin and Greek to create linguistically correct names for genera and species that can be used off the shelf as needed. As proof of principle, we document over a million new names for Bacteria and Archaea. We are confident that our approach provides a road map for how to create new names for decades to come.</div></div></section><section><h2 class="top" id="secst0010" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="https://www.cell.com/action/doSearch?AllField=%22nomenclature%22&amp;ISSN=0966-842X">nomenclature</a></li><li><a href="https://www.cell.com/action/doSearch?AllField=%22Candidatus%22&amp;ISSN=0966-842X">Candidatus</a></li><li><a href="https://www.cell.com/action/doSearch?AllField=%22metagenome-assembled%20genomes%22&amp;ISSN=0966-842X">metagenome-assembled genomes</a></li><li><a href="https://www.cell.com/action/doSearch?AllField=%22genome-based%20taxonomy%22&amp;ISSN=0966-842X">genome-based taxonomy</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="s0005" data-locator="s0005" tabindex="0"><h2 class="top" id="secst0015" data-left-hand-nav="The Legacy of Linnaeus" tabindex="0"><span class="top__sub"></span><span class="top__text">The Legacy of Linnaeus</span></h2><div class="section-paragraph">In the 18th century, the Swedish naturalist Linnaeus proposed a hierarchical scheme of <strong>taxonomy</strong> (see <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#gs0005" data-locator="gs0005" class="scroll-into-link">Glossary</a>) that assigned Latin <strong>binomials</strong> to biological species [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0005" id="back-bb0005" data-db-target-for="bb0005-d8213092e944" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0005">1</a><div data-db-target-of="bb0005-d8213092e944" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linnaeus C. </li></ul><div class="reference-citations__detials">Systema Naturae.<div class="popup-ref">10th edn.  <span class="refPublisherName">Laurentius Salvius</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1759&amp;author=Linnaeus+C.&amp;title=Systema+Naturae">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Shortly afterwards, the first genus names were applied to bacteria [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0010" id="back-bb0010" data-db-target-for="bb0010-d8213092e947" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0010">2</a><div data-db-target-of="bb0010-d8213092e947" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Müller O.F. </li></ul><div class="reference-citations__detials">Animalcula infusoria fluviatilia et marina, quae detexit, systematice descripsit et ad vivum delineari curavit.<div class="popup-ref"> <span class="refPublisherName">Typis Nicolai Mölleri</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1786</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1786&amp;author=M%C3%BCller+O.F.&amp;title=Animalcula+infusoria+fluviatilia+et+marina%2C+quae+detexit%2C+systematice+descripsit+et+ad+vivum+delineari+curavit">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. In the 19th century, Darwin’s <em>Origin of Species</em> provided an evolutionary framework for taxonomy, confidently proclaiming: 'Our classifications will come to be, as far as they can be so made, genealogies' [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0015" id="back-bb0015" data-db-target-for="bb0015-d8213092e954" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0015">3</a><div data-db-target-of="bb0015-d8213092e954" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Darwin C. </li></ul><div class="reference-citations__detials">On the Origin of Species.<div class="popup-ref">1st edn.  <span class="refPublisherName">John Murray</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1859</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1859&amp;author=Darwin+C.&amp;title=On+the+Origin+of+Species">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. In the 20th century, Hennig brought clarity to evolutionary taxonomy through the development of 'phylogenetic systematics', now commonly called <strong>cladistics</strong>, which stipulated that biological classifications must represent the phylogenies of organisms and that taxa should represent <strong>monophyletic groups</strong> [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0020" id="back-bb0020" data-db-target-for="bb0020-d8213092e963" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0020">4</a><div data-db-target-of="bb0020-d8213092e963" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hennig W. </li></ul><div class="reference-citations__detials">Grundzüge einer Theorie der phylogenetischen Systematik.<div class="popup-ref"> <span class="refPublisherName">Deutscher Zentralverlag</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1950</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1950&amp;author=Hennig+W.&amp;title=Grundz%C3%BCge+einer+Theorie+der+phylogenetischen+Systematik">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].</div><div class="section-paragraph">Linnaean binomials, drawing on combinations of <strong>Latin</strong> and <strong>Ancient Greek</strong> roots, have stood the test of time in providing a stable, clear, and memorable system of <strong>nomenclature</strong> across biology. Efforts to codify bacterial nomenclature have culminated in the <strong>International Code of Nomenclature of Prokaryotes</strong> (ICNP or 'the Code') [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0025" id="back-bb0025" data-db-target-for="bb0025-d8213092e980" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0025">5</a><div data-db-target-of="bb0025-d8213092e980" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parker C.T. </li><li>et al.</li></ul><div class="reference-citations__detials">International Code of Nomenclature of Prokaryotes.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">S1-S111</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.000778&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=26596770&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85050686253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (99)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2019&amp;pages=S1-S111&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Parker+C.T.&amp;title=International+Code+of+Nomenclature+of+Prokaryotes">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>], which sets out the rules for naming species of Archaea and Bacteria. The enduring legacy of Linnaeus in microbiology is evident from the remarkable success of the ICNP in overseeing the valid publication of names for over 3400 bacterial and archaeal genera and over 20 000 bacterial and archaeal species. These names are superbly well documented in the <strong>List of Prokaryotic names with Standing in Nomenclature</strong> (<a href="https://www.bacterio.net/" target="_blank">https://www.bacterio.net</a>) [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0030" id="back-bb0030" data-db-target-for="bb0030-d8213092e990" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0030">6</a><div data-db-target-of="bb0030-d8213092e990" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parte A.C. </li><li>et al.</li></ul><div class="reference-citations__detials">List of prokaryotic names with standing in nomenclature (LPSN) moves to the DSMZ.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span> (<span class="refComment">Published online July 23, 2020. https://doi.org/10.1099/ijsem.0.004332</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.004332&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32701423&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85089371956&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Parte+A.C.&amp;title=List+of+prokaryotic+names+with+standing+in+nomenclature+%28LPSN%29+moves+to+the+DSMZ">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].</div></section><section id="s0010" data-locator="s0010" tabindex="0"><h2 class="top" id="secst0020" data-left-hand-nav="Naming the Unnamed Millions" tabindex="0"><span class="top__sub"></span><span class="top__text">Naming the Unnamed Millions</span></h2><div class="section-paragraph">However, there are clearly several shortcomings to the current approach to nomenclature of Archaea and Bacteria. The most obvious is a deficiency in scale. Comparable efforts for eukaryotes document hundreds of thousands of genera and millions of species (<a href="http://www.catalogueoflife.org/annual-checklist/2019/info/about" target="_blank">http://www.catalogueoflife.org/annual-checklist/2019/info/about</a>) [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0035" id="back-bb0035" data-db-target-for="bb0035-d8213092e1001" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0035">7</a><div data-db-target-of="bb0035-d8213092e1001" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rees T. </li><li>et al.</li></ul><div class="reference-citations__detials">All genera of the world: an overview and estimates based on the March 2020 release of the Interim Register of Marine and Nonmarine Genera (IRMNG).<div class="popup-ref"><span class="ref__series"><i>Megataxa.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">123-140</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.11646%2Fmegataxa.1.2.3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2020&amp;pages=123-140&amp;journal=Megataxa&amp;author=Rees+T.&amp;title=All+genera+of+the+world%3A+an+overview+and+estimates+based+on+the+March+2020+release+of+the+Interim+Register+of+Marine+and+Nonmarine+Genera+%28IRMNG%29">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Although estimates of the total number of bacterial and archaeal species vary from millions to billions [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0040" id="back-bb0040" data-db-target-for="bb0040-d8213092e1004" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0040">8</a><div data-db-target-of="bb0040-d8213092e1004" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dykhuizen D.E. </li></ul><div class="reference-citations__detials">Santa Rosalia revisited: why are there so many species of bacteria.<div class="popup-ref"><span class="ref__series"><i>Antonie van Leeuwenhoek.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">25-33</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1023%2FA%3A1000665216662&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=9602276&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0031981808&amp;cf=" onclick="newWindow(this.href);return false">Scopus (215)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=1998&amp;pages=25-33&amp;journal=Antonie+van+Leeuwenhoek&amp;author=Dykhuizen+D.E.&amp;title=Santa+Rosalia+revisited%3A+why+are+there+so+many+species+of+bacteria">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>,<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0045" id="back-bb0045" data-db-target-for="bb0045-d8213092e1007" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0045">9</a><div data-db-target-of="bb0045-d8213092e1007" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Amann R. </li><li class="loa__item">Rosselló-Móra R. </li></ul><div class="reference-citations__detials">After all, only millions.<div class="popup-ref"><span class="ref__series"><i>mBio.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>e00999-16</div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FmBio.00999-16&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=27381294&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-84986593817&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;journal=mBio&amp;author=Amann+R.&amp;author=Rossell%C3%B3-M%C3%B3ra+R.&amp;title=After+all%2C+only+millions">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>], even the most conservative figures amply document an unmet need for many millions of new names for genera and species of Archaea and Bacteria.</div><div class="section-paragraph">Another pressing problem is that most microbiologists follow Shakespeare in possessing, at best, 'small Latin, less Greek' [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0050" id="back-bb0050" data-db-target-for="bb0050-d8213092e1012" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0050">10</a><div data-db-target-of="bb0050-d8213092e1012" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jonson B. </li></ul><div class="reference-citations__detials">To the memory of my beloved, The Author Mr William Shakespeare: And what he hath left us.<div class="popup-ref">in: Mr. William Shakespeares Comedies Histories &amp; Tragedies: Published according to the True Originall Copies.  <span class="refPublisherName">Isaac Iaggard and Edward Blount</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1623</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Jonson+B.+To+the+memory+of+my+beloved%2C+The+Author+Mr+William+Shakespeare%3A+And+what+he+hath+left+us.in%3A+Mr.+William+Shakespeares+Comedies+Histories+%26+Tragedies%3A+Published+according+to+the+True+Originall+Copies.+Isaac+Iaggard+and+Edward+Blount%2C%0A+1623">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>] and so are poorly equipped for creating well-formed binomials that comply with the rules of Latin grammar and are presented with clear, plausible etymological justifications (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#b0005" class="scroll-into-link">Box 1</a>). Despite the publication of several 'how-to' guides [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0055" data-db-target-for="bb0055-d8213092e1019" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0055">11.</a><div data-db-target-of="bb0055-d8213092e1019" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">MacAdoo T.O. </li></ul><div class="reference-citations__detials">Nomenclatural Literacy.<div class="popup-ref">in: <span class="editors">Goodfellow M. </span>Handbook of new bacterial systematics.  <span class="refPublisherName">Academic Press</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1993</span>: <span class="ref__seriesPages">339-358</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=MacAdoo+T.O.+Nomenclatural+Literacy.in%3A+Goodfellow+M.+Handbook+of+new+bacterial+systematics.+Academic+Press%2C%0A+1993%3A+339-358">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0060" data-db-target-for="bb0060-d8213092e1019" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0060">12.</a><div data-db-target-of="bb0060-d8213092e1019" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Trüper H.G. </li></ul><div class="reference-citations__detials">How to name a prokaryote? Etymological considerations, proposals and practical advice in prokaryote nomenclature.<div class="popup-ref"><span class="ref__series"><i>FEMS Microbiol. Rev.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">231-249</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=1999&amp;pages=231-249&amp;journal=FEMS+Microbiol.+Rev.&amp;author=Tr%C3%BCper+H.G.&amp;title=How+to+name+a+prokaryote%3F+Etymological+considerations%2C+proposals+and+practical+advice+in+prokaryote+nomenclature">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0065" data-db-target-for="bb0065-d8213092e1019" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0065">13.</a><div data-db-target-of="bb0065-d8213092e1019" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li></ul><div class="reference-citations__detials">How to name new taxa of Archaea and Bacteria.<div class="popup-ref">in: <span class="editors">Whitman W.B. </span>Bergey’s Manual of Systematics of Archaea and Bacteria.  <span class="refPublisherName">John Wiley</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2015</span>: <span class="ref__seriesPages">1-24</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Oren+A.+How+to+name+new+taxa+of+Archaea+and+Bacteria.in%3A+Whitman+W.B.+Bergey%E2%80%99s+Manual+of+Systematics+of+Archaea+and+Bacteria.+John+Wiley%2C%0A+2015%3A+1-24">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>], this skills gap has led to propagation of numerous erroneous malformations – a high-profile example is the species epithet <em>pyloridis</em>, which even passed validation in the <em>International Journal of Systematic Bacteriology</em> before it had to be corrected, according to the rules of Latin grammar, to <em>pylori</em> [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0070" id="back-bb0070" data-db-target-for="bb0070-d8213092e1032" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0070">14</a><div data-db-target-of="bb0070-d8213092e1032" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anonymous </li></ul><div class="reference-citations__detials">Validation of the publication of new names and new combinations previously effectively published outside the IJSB. List no. 17.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1985; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">223-225</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-35-2-223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0021945642&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=1985&amp;pages=223-225&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Anonymous+&amp;title=Validation+of+the+publication+of+new+names+and+new+combinations+previously+effectively+published+outside+the+IJSB.+List+no.+17">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>,<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0075" id="back-bb0075" data-db-target-for="bb0075-d8213092e1035" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0075">15</a><div data-db-target-of="bb0075-d8213092e1035" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Marshall B.J. </li><li class="loa__item">Goodwin C.S. </li></ul><div class="reference-citations__detials">Revised nomenclature of <em>Campylobacter pyloridis</em>.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1987; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">68</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-37-1-68&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0023120054&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1987&amp;pages=68&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Marshall+B.J.&amp;author=Goodwin+C.S.&amp;title=Revised+nomenclature+of+Campylobacter+pyloridis">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. What’s more, bacteriologists are bound by the provisions of the Code, which include many detailed and difficult rules and recommendations on how names should be formulated [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0025" id="back-bb0025" data-db-target-for="bb0025-d8213092e1038" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0025">5</a><div data-db-target-of="bb0025-d8213092e1038" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parker C.T. </li><li>et al.</li></ul><div class="reference-citations__detials">International Code of Nomenclature of Prokaryotes.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">S1-S111</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.000778&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=26596770&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85050686253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (99)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2019&amp;pages=S1-S111&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Parker+C.T.&amp;title=International+Code+of+Nomenclature+of+Prokaryotes">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. These exacting requirements mean that new names have to undergo time-consuming nomenclatorial quality control by a dwindling pool of experts, who are required to be conversant with classical languages, the Code, and contemporary microbiology [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0080" id="back-bb0080" data-db-target-for="bb0080-d8213092e1041" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0080">16</a><div data-db-target-of="bb0080-d8213092e1041" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li><li>et al.</li></ul><div class="reference-citations__detials">Wanted: microbiologists with basic knowledge of Latin and Greek to join our ‘nomenclature quality control’ team.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">65</span></span>: <span class="ref__seriesPages">3761-3762</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.000663&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=26653899&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-84955579461&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2015&amp;pages=3761-3762&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Wanted%3A+microbiologists+with+basic+knowledge+of+Latin+and+Greek+to+join+our+%E2%80%98nomenclature+quality+control%E2%80%99+team">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. These problems are compounded by the custom of creating names on an ad hoc<em>,</em> as-needed, just-in-time fashion, which provides a non-stop drip-by-drip flow of work for nomenclatural experts.<div class="floatDisplay"><div class="accordion--textbox"><div id="b0005" class="accordion" tabindex="0"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" aria-expanded="false" class="accordion__control" aria-controls="b0005" title="Expand"><span>Box 1</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;"><b>Why It Is Hard to Create Well-Formed Binomials</b><div class="section-paragraph">Latin remains the language of taxonomic nomenclature. This brings the advantages of neutrality and stability, but presents problems for most microbiologists, as Latin is no longer widely taught in schools. Unlike English, Latin is a highly inflected language, where the endings of nouns and adjectives vary according to their role in the sentence and linguistic properties. For example, adjectives change their endings to reflect the gender of the noun they are qualifying, for example, the neuter form <em>faecale</em> is used in <em>Microbacterium faecale</em>, but the masculine form <em>faecalis</em> is used in <em>Enterococcus faecalis.</em> Another problem is that many taxonomic names come from Ancient Greek, which has its own alphabet, and so have to be transcribed into the Roman alphabet and then Latinised before use (experts still argue over whether <em>Acinetobacter</em> should have been <em>Akinetobacter</em>).</div><div class="section-paragraph">These problems are compounded by the fact that bacteriologists are also bound by the Code, which includes 65 rules and dozens of recommendations, some requiring subjective judgements, for example, avoiding names that are 'very long or difficult to pronounce'. The Code insists that words from languages other than Latin or Greek should be avoided if equivalents exist in Latin or Greek, but allows genus names to be created in an arbitrary manner, so long as they are treated as Latin nouns.</div><div class="section-paragraph">All this makes it difficult for the non-expert to get things right, so that up to half of all newly proposed names for Archaea and Bacteria need to be corrected before use. Common problems include trying to use poorly Latinised English words (e.g., <em>geesorum</em> instead of <em>anserum</em> for a species associated with geese) or making up nonsensical etymologies [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0140" id="back-bb0140" data-db-target-for="bb0140-d8213092e727" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0140">28</a><div data-db-target-of="bb0140-d8213092e727" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li></ul><div class="reference-citations__detials">Naming novel prokaryotic taxa discovered in the human gut.<div class="popup-ref"><span class="ref__series"><i>Gut.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">969-970</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1136%2Fgutjnl-2019-318716&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=31053603&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85065150710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2020&amp;pages=969-970&amp;journal=Gut&amp;author=Oren+A.&amp;title=Naming+novel+prokaryotic+taxa+discovered+in+the+human+gut">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. The Code clarifies that names are primarily labels rather than descriptions: 'The primary purpose of giving a name to a taxon is to supply a means of referring to it rather than to indicate the characters or the history of the taxon'. The Code also explains what is required for names to be validly published, which includes a description of the taxon. Valid publication of names is typically accompanied by a protologue, which includes a description of the taxon with an etymology and designation of type material.</div></div></div></div></div></div><div class="section-paragraph">Another key challenge stems from the exhilarating success of high-throughput sequencing and bioinformatics, which, twinned with <strong>molecular phylogenetics</strong>, represent a remarkable unifying force across the whole of biology, drawing together all cellular organisms into a single great tree of life (<a href="http://tolweb.org/tree/" target="_blank">http://tolweb.org/tree/</a>). Within microbiology, such advances have been driven by <strong>culturomics</strong> (high-throughput culture followed by whole-genome sequencing, which has delivered many hundreds of new species) and via metagenomics, which has delivered many thousands of <strong>metagenome-assembled genomes</strong> (<strong>MAGs</strong>), mostly from uncultured organisms [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0085" id="back-bb0085" data-db-target-for="bb0085-d8213092e1065" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0085">17</a><div data-db-target-of="bb0085-d8213092e1065" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lagier J.C. </li><li>et al.</li></ul><div class="reference-citations__detials">Culturing the human microbiota and culturomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Microbiol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">540-550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41579-018-0041-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=29937540&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85048972988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (156)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2018&amp;pages=540-550&amp;journal=Nat.+Rev.+Microbiol.&amp;author=Lagier+J.C.&amp;title=Culturing+the+human+microbiota+and+culturomics">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>,<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0090" id="back-bb0090" data-db-target-for="bb0090-d8213092e1068" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0090">18</a><div data-db-target-of="bb0090-d8213092e1068" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parks D.H. </li><li>et al.</li></ul><div class="reference-citations__detials">Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life.<div class="popup-ref"><span class="ref__series"><i>Nat. Microbiol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">1533-1542</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41564-017-0012-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=28894102&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85029026923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (369)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2017&amp;pages=1533-1542&amp;journal=Nat.+Microbiol.&amp;author=Parks+D.H.&amp;title=Recovery+of+nearly+8%2C000+metagenome-assembled+genomes+substantially+expands+the+tree+of+life">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. In addition, bioinformatics analyses have enabled the development of a comprehensive <strong>genome-based taxonomy</strong>, GTDB, ranging from species up to domains [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0095" id="back-bb0095" data-db-target-for="bb0095-d8213092e1074" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0095">19</a><div data-db-target-of="bb0095-d8213092e1074" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parks D.H. </li><li>et al.</li></ul><div class="reference-citations__detials">A complete domain-to-species taxonomy for Bacteria and Archaea.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">1079-1086</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41587-020-0501-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32341564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85084211665&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2020&amp;pages=1079-1086&amp;journal=Nat.+Biotechnol.&amp;author=Parks+D.H.&amp;title=A+complete+domain-to-species+taxonomy+for+Bacteria+and+Archaea">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].</div><div class="section-paragraph">While nomenclature has largely kept up with culturomics [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0100" id="back-bb0100" data-db-target-for="bb0100-d8213092e1079" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0100">20</a><div data-db-target-of="bb0100-d8213092e1079" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lagier J.C. </li><li>et al.</li></ul><div class="reference-citations__detials">Naming microorganisms: the contribution of the IHU Méditerranée Infection, Marseille, France.<div class="popup-ref"><span class="ref__series"><i>New Microbes New Infect.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">S89-S95</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1016%2Fj.nmni.2018.08.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=30402249&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85052989723&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=S89-S95&amp;journal=New+Microbes+New+Infect.&amp;author=Lagier+J.C.&amp;title=Naming+microorganisms%3A+the+contribution+of+the+IHU+M%C3%A9diterran%C3%A9e+Infection%2C+Marseille%2C+France">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>], valid publication of names for bacterial and archaeal species currently requires deposition of cultured type strains in public repositories. This requirement controversially precludes application of the ICNP rules to uncultured organisms identified and characterised by metagenomics [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0105" id="back-bb0105" data-db-target-for="bb0105-d8213092e1082" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0105">21</a><div data-db-target-of="bb0105-d8213092e1082" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Konstantinidis K.T. </li><li>et al.</li></ul><div class="reference-citations__detials">Uncultivated microbes in need of their own taxonomy.<div class="popup-ref"><span class="ref__series"><i>ISME J.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2399-2406</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fismej.2017.113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=28731467&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85031803725&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2017&amp;pages=2399-2406&amp;journal=ISME+J.&amp;author=Konstantinidis+K.T.&amp;title=Uncultivated+microbes+in+need+of+their+own+taxonomy">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>,<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0110" id="back-bb0110" data-db-target-for="bb0110-d8213092e1085" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0110">22</a><div data-db-target-of="bb0110-d8213092e1085" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Murray A.E. </li><li>et al.</li></ul><div class="reference-citations__detials">Roadmap for naming uncultivated Archaea and Bacteria.<div class="popup-ref"><span class="ref__series"><i>Nat. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">987-994</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41564-020-0733-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32514073&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85086165710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2020&amp;pages=987-994&amp;journal=Nat.+Microbiol.&amp;author=Murray+A.E.&amp;title=Roadmap+for+naming+uncultivated+Archaea+and+Bacteria">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>] (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#b0010" class="scroll-into-link">Box 2</a>). One work-around is to apply the designation <strong><em>Candidatus</em></strong> (abbreviated to <em>Ca.</em>) to names for uncultured taxa [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0115" id="back-bb0115" data-db-target-for="bb0115-d8213092e1100" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0115">23</a><div data-db-target-of="bb0115-d8213092e1100" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Murray R.G. </li><li class="loa__item">Stackebrandt E. </li></ul><div class="reference-citations__detials">Taxonomic note: implementation of the provisional status Candidatus for incompletely described procaryotes.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">186-187</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-45-1-186&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=7857801&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0028887837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (297)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1995&amp;pages=186-187&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Murray+R.G.&amp;author=Stackebrandt+E.&amp;title=Taxonomic+note%3A+implementation+of+the+provisional+status+Candidatus+for+incompletely+described+procaryotes">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Although the resulting names have no standing according to the Code, the <em>Candidatus</em> approach provides a clear, memorable, and potentially stable nomenclature for uncultured species that mirrors the nomenclature for cultured species. However, so far barely more than 850 species-level <em>Candidatus</em> names have been published in the peer-reviewed literature [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0120" id="back-bb0120" data-db-target-for="bb0120-d8213092e1109" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0120">24</a><div data-db-target-of="bb0120-d8213092e1109" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li><li>et al.</li></ul><div class="reference-citations__detials">Lists of names of prokaryotic Candidatus taxa.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">3956-4042</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.003789&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32603289&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85088879544&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2020&amp;pages=3956-4042&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Lists+of+names+of+prokaryotic+Candidatus+taxa">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].<div class="floatDisplay"><div class="accordion--textbox"><div id="b0010" class="accordion" tabindex="0"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" aria-expanded="false" class="accordion__control" aria-controls="b0010" title="Expand"><span>Box 2</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;"><b>Culture Wars</b><div class="section-paragraph">Anyone seeking a stable system of nomenclature would not start from where we are now. Rather than one code for all organisms, there are several, with different rules for naming algae-fungi-and-plants, animals, and Bacteria-and-Archaea. Cyanobacteria were for a long time treated as plants and so most still lack validly published names according to the ICNP. Oddly, names for phyla have never been included in the Code. And, for cladists, the term 'prokaryote' is deprecated – because prokaryotes are no longer considered a monophyletic group [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0145" data-db-target-for="bb0145-d8213092e742" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0145">29.</a><div data-db-target-of="bb0145-d8213092e742" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pace N.R. </li></ul><div class="reference-citations__detials">It’s time to retire the prokaryote.<div class="popup-ref"><span class="ref__series"><i>Microbiol. Today.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">84</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2009&amp;pages=84&amp;journal=Microbiol.+Today&amp;author=Pace+N.R.&amp;title=It%E2%80%99s+time+to+retire+the+prokaryote">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0150" data-db-target-for="bb0150-d8213092e742" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0150">30.</a><div data-db-target-of="bb0150-d8213092e742" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pace N.R. </li></ul><div class="reference-citations__detials">Rebuttal: the modern concept of the procaryote.<div class="popup-ref"><span class="ref__series"><i>J. Bacteriol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">191</span></span>: <span class="ref__seriesPages">2006-2007</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FJB.01296-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=19168604&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-64049096016&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=191&amp;publication_year=2009&amp;pages=2006-2007&amp;journal=J.+Bacteriol.&amp;author=Pace+N.R.&amp;title=Rebuttal%3A+the+modern+concept+of+the+procaryote">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-bb0155" data-db-target-for="bb0155-d8213092e742" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0155">31.</a><div data-db-target-of="bb0155-d8213092e742" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pace N.R. </li></ul><div class="reference-citations__detials">Problems with 'procaryote'.<div class="popup-ref"><span class="ref__series"><i>J. Bacteriol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">191</span> (<span class="refComment">discussion 2011</span>)</span>: <span class="ref__seriesPages">2008-2010</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FJB.01224-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=19168605&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-64049098525&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=191&amp;publication_year=2009&amp;pages=2008-2010&amp;journal=J.+Bacteriol.&amp;author=Pace+N.R.&amp;title=Problems+with+%27procaryote%27">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>] – and so, one could even argue that the ICNP is misnamed and should instead be named the International Code of Nomenclature of Archaea and Bacteria. It is also worth noting that there is no specific term for people who study Archaea and Bacteria – here, we have tended to use 'bacteriologist<em>',</em> noting that the term 'archaeologist' has been appropriated by another discipline.</div><div class="section-paragraph">Despite claiming that 'Nothing in this Code may be construed to restrict the freedom of taxonomic thought or action', the ICNP is very particular about what counts as type material in naming a species: only living pure cultures meet the requirement – an ironic contrast to the International Code of Nomenclature for algae, fungi, and plants, which requires that type material be dead or at least inert! This requirement for live cultures has attracted controversy. One objection is that almost all microorganisms live in communities and in challenging conditions, so organisms grown in pure culture, at best, provide an incomplete view of the natural world – and, at worst, represent a laboratory artefact [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0160" id="back-bb0160" data-db-target-for="bb0160-d8213092e750" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0160">32</a><div data-db-target-of="bb0160-d8213092e750" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hobman J.L. </li><li>et al.</li></ul><div class="reference-citations__detials">Laboratory strains of <em>Escherichia coli</em>: model citizens or deceitful delinquents growing old disgracefully.<div class="popup-ref"><span class="ref__series"><i>Mol. Microbiol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">64</span></span>: <span class="ref__seriesPages">881-885</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1111%2Fj.1365-2958.2007.05710.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=17501914&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-34248373709&amp;cf=" onclick="newWindow(this.href);return false">Scopus (39)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=2007&amp;pages=881-885&amp;journal=Mol.+Microbiol.&amp;author=Hobman+J.L.&amp;title=Laboratory+strains+of+Escherichia+coli%3A+model+citizens+or+deceitful+delinquents+growing+old+disgracefully">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].</div><div class="section-paragraph">A more pressing objection stems from the fact that most microbial species remain uncultured, even though they are increasingly accessible by metagenomics. Many molecular microbiologists now suggest that uncultured organisms should have their own names and that genome sequences should be acceptable as type material [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0110" id="back-bb0110" data-db-target-for="bb0110-d8213092e755" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0110">22</a><div data-db-target-of="bb0110-d8213092e755" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Murray A.E. </li><li>et al.</li></ul><div class="reference-citations__detials">Roadmap for naming uncultivated Archaea and Bacteria.<div class="popup-ref"><span class="ref__series"><i>Nat. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">987-994</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41564-020-0733-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32514073&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85086165710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2020&amp;pages=987-994&amp;journal=Nat.+Microbiol.&amp;author=Murray+A.E.&amp;title=Roadmap+for+naming+uncultivated+Archaea+and+Bacteria">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. However, after lengthy debate, a proposal to amend the Code to accommodate this request was rejected in March 2020 [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0165" id="back-bb0165" data-db-target-for="bb0165-d8213092e758" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0165">33</a><div data-db-target-of="bb0165-d8213092e758" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sutcliffe I.C. </li><li>et al.</li></ul><div class="reference-citations__detials">Minutes of the International Committee on Systematics of Prokaryotes online discussion on the proposed use of gene sequences as type for naming of prokaryotes, and outcome of vote.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">4416-4417</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.004303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32628106&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85088880937&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2020&amp;pages=4416-4417&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Sutcliffe+I.C.&amp;title=Minutes+of+the+International+Committee+on+Systematics+of+Prokaryotes+online+discussion+on+the+proposed+use+of+gene+sequences+as+type+for+naming+of+prokaryotes%2C+and+outcome+of+vote">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].</div><div class="section-paragraph">In the meantime, we have to muddle through with <em>Candidatus</em> names, which, although mentioned in the Code, have no priority. Does this matter? Probably not – as although the <em>de jure</em> position is that a <em>Candidatus</em> name could be replaced by a fresh name any time in the future, the de facto situation is that, for most microbiologists, it will be simply too much effort to create and validly publish new names when perfectly good names already exist. After all, nearly half of the new names assigned to cultured organisms in journals other than the <em>International Journal of Systematic and Evolutionary Microbiology</em> are never validly published, which simply requires a request for inclusion in a Validation List in that journal [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0170" id="back-bb0170" data-db-target-for="bb0170-d8213092e775" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0170">34</a><div data-db-target-of="bb0170-d8213092e775" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li><li>et al.</li></ul><div class="reference-citations__detials">Why are so many effectively published names of prokaryotic taxa never validated.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">2125-2129</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.002851&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=29873629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85049891496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=68&amp;publication_year=2018&amp;pages=2125-2129&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Why+are+so+many+effectively+published+names+of+prokaryotic+taxa+never+validated">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Reassuringly, online resources, such as the List of Prokaryotic Names with Standing in Nomenclature (LPSN), National Centre for Biotechnology Information (NCBI), and GTDB, already incorporate <em>Candidatus</em> names, and such de facto arrangements do a great job at allowing these names to be used while also preventing confusion over which names have already been used, whether formally or informally.</div></div></div></div></div></div><div class="section-paragraph">Similarly, Latin names have yet to be assigned to the vast majority of new species or genera defined by genome-based taxonomies, which include not only those represented solely by MAGs, but also new taxa for which cultured strains are available. Instead, almost all new genera and species identified in these settings have been assigned unstable, confusing, and hard-to-remember alphanumerical identifiers. For example, the current release of the GTDB (<a href="https://gtdb.ecogenomic.org/" target="_blank">https://gtdb.ecogenomic.org</a> R05-RS95 July 17 2020) documents 31 910 species of Archaea and Bacteria, but 23 171 have only placeholder names, including 4827 species epithets created by the addition of an alphabetical suffix to an existing name (e.g., <em>Helicobacter pylori_A</em>). Similarly, of GTDB’s 9428 genera, 6652 have only placeholder names, 708 of them created by appending an alphabetical suffix (e.g., <em>Pseudomonas_D</em>).</div><div class="section-paragraph">So, should we conclude that the legacy of Linnaeus is no longer relevant to microbiology in the age of genomes and metagenomes? Should we be happy to refer to a new species as, for example, UBA6965 or sp000063525? We believe that the answer is a resounding 'no!' However, high-throughput generation of taxa via sequence-based approaches clearly precludes the detailed attention usually applied to the one-by-one construction of Latin binomials. Instead, we propose that the problem can best be solved by automating the creation of well-formed names.</div></section><section id="s0015" data-locator="s0015" tabindex="0"><h2 class="top" id="secst0025" data-left-hand-nav="Automating the Creation of Names" tabindex="0"><span class="top__sub"></span><span class="top__text">Automating the Creation of Names</span></h2><div class="section-paragraph">To meet the need for a stable, clear. and memorable nomenclature for the next million bacterial or archaeal species, we propose abandoning the current cottage-industry approach and instead advocate the automated creation of names en masse, in advance of the need to allocate them to biological entities.</div><div class="section-paragraph">How is this even possible? The answer is an approach that reaches back to classical times: joining individual word roots from Greek or Latin together to create compound words with new meanings. For example, the 1st century Greek geographer Strabo gave us <em>Rhinoceros</em>, combining Ancient Greek roots for 'nose' and 'horn', while Linnaeus named the genus <em>Chrysanthemum</em> using the Ancient Greek roots for 'gold' and 'flower'. This principle has been systematised in the Code to create new genus names, with the rule that a connecting vowel -o- is used after Greek roots and -i- after Latin roots. Any new genus name that is created inherits its grammatical properties (including gender and declension) only from the last element in the word formation. As many as four roots have been combined to give us validly published genus names such as <em>Ectothiorhodospira</em> from the Greek roots for 'outside-sulfur-rose-spiral' or <em>Allocatelliglobosispora</em> from the Greek and Latin roots for 'another-chainlet-sphere-spore'.</div><section id="s0020" data-locator="s0020" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">The Great Automatic Nomenclator</span></h3><div class="section-paragraph">We propose to extend this approach so that very large numbers of genus names can be created using an automated combinatorial concatenation of a relatively small set of starting terms. Let us say we wish to explore a biome-specific generic namespace defined by combinations of three terms (<a data-link="modal" data-target="#image-S0966-842X(20)30271-7gr1" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#f0005" data-locator="f0005" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>, Key Figure). If we select ten roots to be deployed in each of the initial, middle, and final positions, then it becomes possible to create, from just 30 roots, ten-times-ten-times-ten = a thousand names with little effort – an approach we have already used to create names for several hundred new genera from the chicken gut microbiome [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0125" id="back-bb0125" data-db-target-for="bb0125-d8213092e1153" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0125">25</a><div data-db-target-of="bb0125-d8213092e1153" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gilroy R. </li><li>et al.</li></ul><div class="reference-citations__detials">A genomic blueprint of the chicken gut microbiome.<div class="popup-ref"><span class="ref__series"><i>Res. Square.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span> (<span class="refComment">Published online August 17, 2020. http://dx.doi.org/10.21203/rs.3.rs-56027/v1</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;journal=Res.+Square&amp;author=Gilroy+R.&amp;title=A+genomic+blueprint+of+the+chicken+gut+microbiome">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>].<div class="floatDisplay"><figure id="f0005" class="" tabindex="0"><img class="b-lazy b-loaded ux-lazyloaded" alt="Figure 1" data-locator="f0005" data-high-res="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" data-large-image="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gr1.jpg"><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Key Figure. Combinatorial Approaches to the Creation of New Genus Names for Archaea and Bacteria.</span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="https://www.cell.com/action/downloadFigures?pii=S0966-842X(20)30271-7&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">To automate this approach, we have created a Python script named the 'Great Automatic Nomenclator' (or Gan: <a href="https://github.com/telatin/gan" target="_blank">https://github.com/telatin/gan</a>) after a short story by Roald Dahl [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0130" id="back-bb0130" data-db-target-for="bb0130-d8213092e1161" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0130">26</a><div data-db-target-of="bb0130-d8213092e1161" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dahl R. </li></ul><div class="reference-citations__detials">The great automatic grammatizator.<div class="popup-ref">in: <em>Someone Like You</em> Alfred.  <span class="refPublisherName">A. Knopf</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1953</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Dahl+R.+The+great+automatic+grammatizator.in%3A+Someone+Like+You+Alfred.+A.+Knopf%2C%0A+1953">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>], or 'garden' in Hebrew, reflecting its fertile productivity. The script takes, as its input, tables of roots in a specified format and then performs combinatorial concatenation, considering ICNP rules governing the use or elision of connecting vowels. In addition, because the input roots have already completed linguistic quality control, the new names are grammatically correct and come complete with etymological justifications that can be used in a <strong>protologue</strong>. Gan can be interpreted and run using the programming language Python, which can be installed on a range of operating systems (Windows, MacOS, Linux, and even mobile devices running iOS or Android). However, version 1.0 still requires detailed curation of the input files and produces rather basic outputs that can be finessed by editing in, for example, Excel. However, we anticipate that the program will become more user-friendly and productive in subsequent versions.</div></section><section id="s0025" data-locator="s0025" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">The Power of Prefixes</span></h3><div class="section-paragraph">Before exploring the full power of this combinatorial approach, let us take a quick look at an easy win in creating new names that reflect phylogenetic positions. Since the time of Linnaeus, when advances in taxonomy demand that a new taxon be split from an existing taxon, it has been common practice to add a short prefix (or less commonly, a suffix) to the existing name to create a new name, with an <strong>etymology</strong> that defines the new taxon as 'related to but distinct from the' pre-existing taxon. This approach has already seen extensive use for names for Bacteria (<a href="https://lpsn.dsmz.de/text/genera-named-after-other-genera" target="_blank">https://lpsn.dsmz.de/text/genera-named-after-other-genera</a>), using prefixes such as <em>neo-</em> (from the Greek for 'new') or <em>allo-</em> (from the Greek for 'other').<div class="floatDisplay"><div id="t0005" class="inline-table" tabindex="0"><div class="inline-table__head"><span class="inline-table__label">Table 1</span><span class="inline-table__title">Prefixes Used to Create New Genus or Species Names</span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered"><thead><tr><th class="rowsep">Prefix</th><th class="rowsep">Language</th><th class="rowsep">Part of speech</th><th class="rowsep">Classical term</th><th class="rowsep">Definition</th></tr></thead><tbody><tr><td><em>Alii</em></td><td>Latin</td><td>Adjective</td><td><em>alius</em></td><td>Other</td></tr><tr><td><em>Allo</em></td><td>Greek</td><td>Adjective</td><td>ἀλλος</td><td>Other</td></tr><tr><td><em>Alteri</em></td><td>Latin</td><td>Adjective</td><td><em>alter</em></td><td>The other</td></tr><tr><td><em>Amphi</em></td><td>Greek</td><td>Preposition</td><td>ἀμφί</td><td>Around, near</td></tr><tr><td><em>Cognati</em></td><td>Latin</td><td>Adjective</td><td><em>cognatus</em></td><td>Related</td></tr><tr><td><em>Crypto</em></td><td>Greek</td><td>Adjective</td><td>κρυπτός</td><td>Hidden</td></tr><tr><td><em>Enantio</em></td><td>Greek</td><td>Adjective</td><td>ἐνᾰντῐ́ος</td><td>Opposite to</td></tr><tr><td><em>Epi</em></td><td>Greek</td><td>Preposition</td><td>ἐπί</td><td>On top of</td></tr><tr><td><em>Extra</em></td><td>Latin</td><td>Preposition</td><td><em>extra</em></td><td>Beyond</td></tr><tr><td><em>Falsi</em></td><td>Latin</td><td>Adjective</td><td><em>falsus</em></td><td>False</td></tr><tr><td><em>Hetero</em></td><td>Greek</td><td>Adjective</td><td>ἕτερος</td><td>Different</td></tr><tr><td><em>Hosper</em></td><td>Greek</td><td>Adverb</td><td>ὥσπερ</td><td>Like</td></tr><tr><td><em>Hyper</em></td><td>Greek</td><td>Preposition</td><td>ὑπέρ</td><td>Over, beyond</td></tr><tr><td><em>Iso</em></td><td>Greek</td><td>Adjective</td><td><em>ἴσος</em></td><td>Same as</td></tr><tr><td><em>Juxta</em></td><td>Latin</td><td>Adjective</td><td><em>iuxta</em></td><td>Nearby</td></tr><tr><td><em>Meso</em></td><td>Greek</td><td>Adjective</td><td>μέσος</td><td>Middle</td></tr><tr><td><em>Meta</em></td><td>Greek</td><td>Preposition</td><td>μετᾰ́</td><td>Besides</td></tr><tr><td><em>Neo</em></td><td>Greek</td><td>Adjective</td><td>νέος</td><td>New</td></tr><tr><td><em>Notho</em></td><td>Greek</td><td>Adjective</td><td>νόθος</td><td>Crossbred</td></tr><tr><td><em>Novi</em></td><td>Latin</td><td>Adjective</td><td><em>novus</em></td><td>New</td></tr><tr><td><em>Paeni</em></td><td>Latin</td><td>Adverb</td><td><em>paene</em></td><td>Almost</td></tr><tr><td><em>Para</em></td><td>Greek</td><td>Preposition</td><td>παρά</td><td>Beside</td></tr><tr><td><em>Peri</em></td><td>Greek</td><td>Preposition</td><td>περί</td><td>About</td></tr><tr><td><em>Praeter</em></td><td>Latin</td><td>Preposition</td><td><em>praeter</em></td><td>Beyond</td></tr><tr><td><em>Prope</em></td><td>Latin</td><td>Preposition</td><td><em>prope</em></td><td>Near</td></tr><tr><td><em>Pseudo</em></td><td>Greek</td><td>Adjective</td><td>ψευδής</td><td>False</td></tr><tr><td><em>Quasi</em></td><td>Latin</td><td>Conjunction</td><td><em>quasi</em></td><td>As if</td></tr><tr><td><em>Simili</em></td><td>Latin</td><td>Adjective</td><td><em>similis</em></td><td>Similar</td></tr><tr><td><em>Tele</em></td><td>Greek</td><td>Preposition</td><td>τῆλε</td><td>Far away</td></tr><tr><td><em>Ultra</em></td><td>Latin</td><td>Preposition</td><td><em>ultra</em></td><td>Beyond</td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="https://www.cell.com/action/showFullTableHTML?isHtml=true&amp;tableId=t0005&amp;pii=S0966-842X%2820%2930271-7">
                            Open table in a new tab
                        </a></li></ul></div></div></div><div class="section-paragraph">We have collated a list of 38 prefixes that can be used for this purpose (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#t0005" id="back-t0005" class="scroll-into-link">Table 1</a>). We then used Gan to apply these prefixes to validly published names for Bacteria and Archaea. Using this approach, we have been able to create over 130 000 new genus names and over 700 000 species names, complete with grammatical metadata and etymological justifications, that can be applied to sister taxa related to, but distinct from, already named taxa (see <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0015" data-locator="ec0015" class="scroll-into-link">Table S3</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0020" data-locator="ec0020" class="scroll-into-link">Table S4</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0025" data-locator="ec0025" class="scroll-into-link">Table S5</a> in the supplemental information online). Of course, some of these names will never be used as the number of new names is much larger than the number of new sister taxa that are likely to be discovered. However, as proof of immediate utility, this approach could be applied to all genera marked in GTDB simply with an alphabetical suffix (<em>Bacillus_A</em>, <em>Bacillus_B</em> etc.) to generate well-formed Latin names for over 600 new genera. It is also worth noting that there are precedents for incorporating more than one prefix into a bacterial name (e.g., <em>Parapseudoflavitalea</em> or <em>Allopseudarcicella)</em>, so if we allow two prefixes to be added to all existing names (while avoiding using the same prefix twice) we would be able to generate over 4 million new genus names and 29 million new species names.</div></section><section id="s0030" data-locator="s0030" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">Flexible Endings</span></h3><div class="section-paragraph">Often in the past, final word elements for bacterial genus names have reflected cellular morphology – as in the ending '<em>coccus</em>' in <em>Enterococcus</em>, describing a coccus associated with the gut. However, if we are to create a set of names that can be applied flexibly to any bacterium or archaeon, particularly to uncultured genera, we need to use last-word elements that can be used without knowledge of phenotypic characters (e.g., cellular or colonial morphology). We have therefore collated a set of last-word elements that can be used in genus names derived from biomes and/or in association with proper nouns (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#t0010" id="back-t0010" class="scroll-into-link">Table 2</a>). Use of such elements brings not just remarkable combinatorial power, but also minimises clashes with botanical and zoological codes.<div class="floatDisplay"><div id="t0010" class="inline-table" tabindex="0"><div class="inline-table__head"><span class="inline-table__label">Table 2</span><span class="inline-table__title">Final Word Elements for Bacterial or Archaeal Genera<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0005" data-db-target-for="tf0005-d8213092e432" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0005"><sup>a</sup></a><div data-db-target-of="tf0005-d8213092e432" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>a </span><span>Abbreviations: fem., feminine; L., Latin; masc., masculine; neut., neuter; N.L., New Latin.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></div><div class="table-with-References"> <table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered"><thead><tr><th class="rowsep">Application</th><th class="rowsep">Element</th><th class="rowsep">Language</th><th class="rowsep">Part of speech</th><th class="rowsep">Gender</th><th class="rowsep">Definition</th></tr></thead><tbody><tr><td>Biomes</td><td><em>Adaptatus</em></td><td>L.</td><td>Adjectival noun</td><td>Masc.</td><td>Adapted to</td></tr><tr><td>Biomes</td><td><em>Cola</em></td><td>N.L.</td><td>Suffix</td><td>Masc./fem.<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0010" data-db-target-for="tf0010-d8213092e485" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0010"><sup>b</sup></a><div data-db-target-of="tf0010-d8213092e485" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>b </span><span>Although either gender can be used, GAN makes it feminine by default.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>An inhabitant of</td></tr><tr><td>Biomes</td><td><em>Enecus</em></td><td>N.L.</td><td>Noun</td><td>Masc.</td><td>Inhabitant</td></tr><tr><td>Biomes</td><td><em>Habitans</em></td><td>L.</td><td>Adjectival noun</td><td>Masc./fem.<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0010" data-db-target-for="tf0010-d8213092e516" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0010"><sup>b</sup></a><div data-db-target-of="tf0010-d8213092e516" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>b </span><span>Although either gender can be used, GAN makes it feminine by default.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>An inhabitant</td></tr><tr><td>Biomes</td><td><em>Vicinum</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>A neighbour</td></tr><tr><td>Biomes</td><td><em>Vivens</em></td><td>N.L.</td><td>Adjectival noun</td><td>Masc./fem.<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0010" data-db-target-for="tf0010-d8213092e547" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0010"><sup>b</sup></a><div data-db-target-of="tf0010-d8213092e547" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>b </span><span>Although either gender can be used, GAN makes it feminine by default.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>Living</td></tr><tr><td>Biomes/proper names</td><td><em>Archaeum</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>An archaeon<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0015" data-db-target-for="tf0015-d8213092e566" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0015"><sup>c</sup></a><div data-db-target-of="tf0015-d8213092e566" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>c </span><span>To be used only for Archaea or Bacteria, respectively.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td></tr><tr><td>Biomes/proper names</td><td><em>Bacterium</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>A bacterium<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0015" data-db-target-for="tf0015-d8213092e583" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0015"><sup>c</sup></a><div data-db-target-of="tf0015-d8213092e583" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>c </span><span>To be used only for Archaea or Bacteria, respectively.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td></tr><tr><td>Biomes/proper names</td><td><em>Microbium</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>A microbe</td></tr><tr><td>Biomes/proper names</td><td><em>Monas</em></td><td>N.L.</td><td>Noun</td><td>Fem.</td><td>A monad</td></tr><tr><td>Biomes/proper names</td><td><em>Morpha</em></td><td>N.L.</td><td>Noun</td><td>Fem.</td><td>Form, shape</td></tr><tr><td>Biomes/proper names</td><td><em>Ousia</em></td><td>N.L.</td><td>Noun</td><td>Fem.</td><td>Essence</td></tr><tr><td>Biomes/proper names</td><td><em>Plasma</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>A form</td></tr><tr><td>Biomes/proper names</td><td><em>Soma</em></td><td>N.L.</td><td>Noun</td><td>Neut.</td><td>A body</td></tr></tbody></table></div><div class="inline-table__tail"><div class="inline-table__footnote"><span>a </span><span>Abbreviations: fem., feminine; L., Latin; masc., masculine; neut., neuter; N.L., New Latin.</span></div><div class="inline-table__footnote"><span>b </span><span>Although either gender can be used, GAN makes it feminine by default.</span></div><div class="inline-table__footnote"><span>c </span><span>To be used only for Archaea or Bacteria, respectively.</span></div></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="https://www.cell.com/action/showFullTableHTML?isHtml=true&amp;tableId=t0010&amp;pii=S0966-842X%2820%2930271-7">
                            Open table in a new tab
                        </a></li></ul></div></div></div></section><section id="s0035" data-locator="s0035" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">People and Places</span></h3><div class="section-paragraph">Under the auspices of the ICNP, Archaea and Bacteria have often been named after places or people (mythical or real). In 2005, the nomenclature expert Hans Trüper expressed exasperation at excessive use of this approach with place names, which he termed 'localimania'. However, the practice continues, with the most salient example being the use of <em>Massilia</em> (the Latin name for Marseille) by the IHU Méditerranée Infection – a term that has found its way into over 260 species or genus names [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0100" id="back-bb0100" data-db-target-for="bb0100-d8213092e1246" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0100">20</a><div data-db-target-of="bb0100-d8213092e1246" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lagier J.C. </li><li>et al.</li></ul><div class="reference-citations__detials">Naming microorganisms: the contribution of the IHU Méditerranée Infection, Marseille, France.<div class="popup-ref"><span class="ref__series"><i>New Microbes New Infect.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">S89-S95</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1016%2Fj.nmni.2018.08.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=30402249&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85052989723&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=S89-S95&amp;journal=New+Microbes+New+Infect.&amp;author=Lagier+J.C.&amp;title=Naming+microorganisms%3A+the+contribution+of+the+IHU+M%C3%A9diterran%C3%A9e+Infection%2C+Marseille%2C+France">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Usefully, these include validly published precedents for combining a proper noun with other roots, for example, <em>Methanomassiliicoccus</em>. The way is thus open for combining names of places associated with identification of new taxa through genomic or metagenomic analyses with additional roots, including our set of last-word elements, for example, <em>Brisbanmonas, Brisbanibacterium,</em> for species delineated by the GTDB project in Brisbane (<a data-link="modal" data-target="#image-S0966-842X(20)30271-7gr1" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#f0005" data-locator="f0005" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>).</div><div class="section-paragraph">Linnaeus made widespread use of the names drawn from mythology. This practice continues in microbiology. For example, the genus name <em>Cronobacter</em> was applied to a pathogen of children after Cronos, a Titan who swallowed his children as soon as they were born. Again, this approach provides a precedent for combining a proper noun with other roots in, for example, <em>Neptunicoccus</em> or <em>Poseidonocella</em>, and paves the way for the creation of names for new taxa identified through genomic or metagenomic analyses. For example, combining our flexible end elements with names for over 100 sea deities drawn from diverse cultures, we have been able to create names for over 1000 marine microorganisms (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>).</div><div class="section-paragraph">The ICNP provides rules for a well established approach for turning surnames into genus names by the addition of Latin endings or diminutives – examples include <em>Escherichia</em> and <em>Salmonella.</em> Recently, this approach has been broadened into combining personal names with other roots, but so far only for a couple of dozen names [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0135" id="back-bb0135" data-db-target-for="bb0135-d8213092e1291" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0135">27</a><div data-db-target-of="bb0135-d8213092e1291" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Oren A. </li><li>et al.</li></ul><div class="reference-citations__detials">Formation of compound generic names based on personal names: a proposal for emendation of Appendix 9 of the International Code of Nomenclature of Prokaryotes.<div class="popup-ref"><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">594-596</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.003192&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=30570480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85061139837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2019&amp;pages=594-596&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Formation+of+compound+generic+names+based+on+personal+names%3A+a+proposal+for+emendation+of+Appendix+9+of+the+International+Code+of+Nomenclature+of+Prokaryotes">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Application of this approach to the several hundred surnames that have already been used in genus names for bacteria and archaea would allow the creation of many thousands of new names (e.g., <em>Salmoniimonas</em>, <em>Salmoniiplasma</em>, <em>Salmoniimicrobium</em>). However, we note that many of those who created the conceptual and technical framework for microbial taxonomy have yet to be honoured in our discipline – for example, you will look in vain for Archaea or Bacteria named after Carl Linnaeus, Charles Darwin, or Willi Hennig! We have therefore compiled a gender-balanced list of 80 worthy scientists and have used our program to create 640 new names from this list, including <em>Darwiniibacterium</em> and <em>Hennigiimonas</em> (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>).</div></section><section id="s0040" data-locator="s0040" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">Binomials for Biomes</span></h3><div class="section-paragraph">Another well established approach for naming new taxa is to describe the habitat or biome in which the organism is found. For example, as we have noted, <em>Enterococcus</em> describes a coccus found in the gut. However, in the age of metagenomics and microbiome research, we need new genus names for inhabitants of each microbiome by the dozen or even in the hundreds.</div><div class="section-paragraph">Fortunately, this need can be easily met using our combinatorial approach to link our final word elements to terms that describe an organ or a host – for example, <em>Enteromonas</em>, for an microbe associated with the intestine, or <em>Avimonas</em> for one associated with birds. As the common names from classical languages for organs or animals typically provide multiple roots for the same organ/tissue (e.g., faeci<em>-,</em> merdi<em>,</em> excrementi<em>,</em> stercori-<em>,</em> cacco- for faeces) or for the same animal (e.g., galli-, pulli<em>-</em>, alectryo-, cotto- for chicken), this approach has allowed us to generate thousands of names for new genera from animal microbiomes, drawing on over 200 curated roots specifying organs, tissues, or hosts (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>). The same approach can be applied to the use of classical roots for biomes associated with plants, for example, <em>Leguminimicrobium</em> for a microbe from beans, or with the abiotic environment, and, for example, <em>Oceanimonas</em> for a microbe from the oceans, or <em>Chthonomicrobium</em> for a subterranean microbe (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>).</div><div class="section-paragraph">This combinatorial approach proves particularly powerful when, as well as using common names from classical languages, one exploits the genus name of the host as a neo-Latin term that can combined with other roots, for example, <em>Drosophilimonas</em> or <em>Arabidopsidimicrobium.</em> As there are hundreds of thousands of named genera of eukaryotes, this opens up the creation of millions of names for host-associated bacterial genera. Similarly, adopting neo-Latinised versions of technical terms for a particular biome, for example, generating roots such as <em>nasopharyngo-, lotici-</em>, <em>bioreactori- or phylloplani</em>-, brings added precision and enhanced fecundity to the creation of names for the inhabitants of microbiomes<em>.</em></div></section><section id="s0045" data-locator="s0045" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">Stepping up to Three Roots</span></h3><div class="section-paragraph">The remarkable power of combinatorial concatenation steps up a gear when we move from two roots in a row to three. Here, we propose an approach in which the first root specifies a general context, for example, a host, a general environment, a person or a place<em>,</em> while the second root specifies a more specific context, such as an organ or tissue or a specific environment. Using our software on terms for animal hosts and their organs/tissues together with our final word elements, we have generated over 100 000 new genus names for inhabitants of animal microbiomes (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>). This approach could also be used for biomes from the abiotic environment, for example, giving us <em>Chthonohydromonas</em> for a microbe from a subterranean water source. However, even more names can be created if existing host genus names or personal nouns are used in the first root position, for example, <em>Triticirhizomicrobium</em>, <em>Darwiniintestimonas</em>, or <em>Brisbaniiterriplasma</em>.</div></section><section id="s0050" data-locator="s0050" tabindex="0"><h3 class="sectionTitle"><span class="top__sub">&nbsp;</span><span class="top__text">The Species Problem</span></h3><div class="section-paragraph">So far, we have concentrated on the creation of genus rather than species names. Nonetheless, a similar principle of combinatorial concatenation of classical roots works here too, even though the context is slightly different. For a start, a species name consists of a genus name and a species epithet. Unlike genus names, species epithets are typically genitive nouns or adjectives, although nouns in the nominative case in apposition are occasionally used. This creates an exacting requirement for selecting the correct form of the noun in the genitive case or of the adjective in the nominative case, which has to take the same gender as the genus name. This has become easier lately, as the requisite forms for such nouns and adjectives can be found in online dictionaries, such as Wiktionary (<a href="https://www.wiktionary.org/" target="_blank">https://www.wiktionary.org</a>). However, as the grammatical properties of a species epithet are inherited from the final component of the word, so as long as that final term is formatted appropriately, it can be used effortlessly and repeatedly in multiple constructions. Furthermore, unlike a genus name or a Linnaean binomial, a species epithet need not be unique and can in fact be used again and again – for example, the epithet <em>massiliensis</em> has been used over 100 times. Thus, as typically only a few dozen species names are needed per genus, a preformed stock of names can easily be created for each biome by reusing single roots across multiple genera (e.g., <em>avium</em>, <em>gallinarum</em>, for species associated generally with birds or specifically with chickens) or by adding multiple roots in front of a final element (e.g., <em>merdavium, faecavium, caccavium</em> for species associated with bird faeces).</div></section></section><section id="s0055" data-locator="s0055" tabindex="0"><h2 class="top" id="secst0065" data-left-hand-nav="Concluding Remarks and Future Perspectives" tabindex="0"><span class="top__sub"></span><span class="top__text">Concluding Remarks and Future Perspectives</span></h2><div class="section-paragraph">In 1999, the nomenclature expert Hans Trüper claimed 'in view of the million names that will have to be formed in the future… [arbitrary names] are a simple necessity, whether Latin formalists like them or not.' [<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#bb0060" id="back-bb0060" data-db-target-for="bb0060-d8213092e1503" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bb0060">12</a><div data-db-target-of="bb0060-d8213092e1503" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Trüper H.G. </li></ul><div class="reference-citations__detials">How to name a prokaryote? Etymological considerations, proposals and practical advice in prokaryote nomenclature.<div class="popup-ref"><span class="ref__series"><i>FEMS Microbiol. Rev.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">231-249</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=1999&amp;pages=231-249&amp;journal=FEMS+Microbiol.+Rev.&amp;author=Tr%C3%BCper+H.G.&amp;title=How+to+name+a+prokaryote%3F+Etymological+considerations%2C+proposals+and+practical+advice+in+prokaryote+nomenclature">Google Scholar</a></li></ul></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>]. Contrary to Trüper, here we have shown how combinatorial use of Greek and Latin roots could be used to create millions of well-formed taxonomic names for Bacteria and Archaea. What’s more, we have put this principle into practise in the documentation of a million names in the supplementary material (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0005" data-locator="ec0005" class="scroll-into-link">Table S1</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0010" data-locator="ec0010" class="scroll-into-link">Table S2</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0015" data-locator="ec0015" class="scroll-into-link">Table S3</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0020" data-locator="ec0020" class="scroll-into-link">Table S4</a>, <a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0025" data-locator="ec0025" class="scroll-into-link">Table S5</a> in the supplemental information online).</div><div class="section-paragraph">In so doing, we have outlined a scalable system for filling taxonomic namespace that circumvents onerous and expert-dependent one-by-one creation of names – exploiting computational automation to deliver millions of names that are linguistically correct, meet the requirements of the ICNP, and so can be used off the shelf, as needed. We are thus providing added impetus to efforts to create a nomenclature for uncultured organisms and hold a mirror up to the current failure to incorporate uncultured organisms into the Code. We expect that our approach could be broadened to cover the need for well-formed names across the whole of the Darwin tree of life (<a href="https://www.darwintreeoflife.org/" target="_blank">https://www.darwintreeoflife.org</a>). We have started a process that raises many questions (see Outstanding Questions) and have created a program that has to be run over the command line, but we predict that, one day, naming Bacteria and Archaea might be as easy as using Google Translate. In the meantime, we have provided a template showing how input files for Gan should be formatted (<a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#ec0030" data-locator="ec0030" class="scroll-into-link">Table S6</a>).</div><div class="section-paragraph">The software we have created for this purpose is freely available. However, it comes with the warning: 'Caveat Nomenclator!' in that it will concatenate terms that simply do not belong together. For example, <em>Gallidentimonas</em> might appear to be a well-formed Latin name, but it is nonsensical as hens do not have teeth. Similarly, some newly created names might fall foul of the ICNP recommendation to avoid names that are overly long, difficult to pronounce or are disagreeable in form (e.g., with prefixes repeated in tandem as in, say, <em>neoneoaurum</em>), so we recommend sorting names by size and using the shortest first and weeding out repetitive forms). In addition, as the current version of the software does not check whether a name has already been listed by LPSN or used under any of the other nomenclature codes, users should check this themselves.</div><div class="section-paragraph">We must also stress that we have not created or even named any new taxa, merely provided software to generate names that could be used for this purpose – but only once they have been published in peer-reviewed journals and have been properly attached to nomenclatural types. For the time being, our names remain naked, as what the jargon calls <strong><em>nomina nuda</em></strong>! The challenge now is for readers in the microbiology community to clothe them with strains, sequences, circumscriptions, positions, and ranks.<div class="accordion--textbox"><div id="b0015" class="accordion" tabindex="0"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" aria-expanded="true" class="accordion__control expanded js--open" aria-controls="b0015" title="Collapse"><span>Outstanding Questions</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: block;"><div class="section-paragraph">How far can the creation of taxonomic names be automated? Will it soon prove possible for bacteriologists to feed in terms in any modern language to a completely automated and unsupervised system to generate any taxonomic names that they need?</div><div class="section-paragraph">Can we replace expert opinion? Should bacteriologists work with colleagues in the digital humanities to make this happen? Or do we need to educate ourselves in Latin and Greek vocabulary and grammar?</div><div class="section-paragraph">Is a reliance on two dead European languages an exercise in neo-colonialism? Should we encourage the use of terms from languages other than Greek or Latin? Or does current practice ensure stability and elegance in nomenclature?</div><div class="section-paragraph">How far can this approach be broadened beyond bacteriology? Could it work with plants and animals? Could it work with protists? Could it be adapted for use with the Earth BioGenome project or the UK’s Darwin Tree of Life project?</div><div class="section-paragraph">What is standing in the way of naming new genera and species identified by genome and metagenome analyses? Do we need to educate and convince those working in microbial genomics and metagenomics? Are MAGs worth naming? Can we couple automated generation of names to automated phylogenomic analyses?</div><div class="section-paragraph">How should we provide uncultured taxa with well-formed names (which might be validated in the future, pending modification of the rules of the Code)? Is the <em>Candidatus</em> approach sufficient or is more needed? How do we incentivise deposition of strains in repositories if sequences are sufficient as type material?</div><div class="section-paragraph">How do we ensure that new names are applied appropriately and uniquely tied to newly discovered taxa?</div></div></div></div></div></section><section><h2 class="top" id="secst0070" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">M.J.P. is supported by the Quadram Institute Bioscience Biotechnology and Biological Sciences Research Council (BBSRC)-funded  Strategic Program: Microbes in the Food Chain  (project no.  BB/R012504/1 ) and its constituent project  BBS/E/F/000PR10351  (Theme 3, Microbial Communities in the Food Chain). A.T. gratefully acknowledges the support of  BBSRC ; this research was funded by the  BBSRC Institute Strategic Programme Gut Microbes and Health   BB/R012490/1  and its constituent project  BBS/E/F/000PR10353 .</div></section><section id="s0060" data-locator="s0060" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental Information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental Information</span></h2><div class="section-paragraph supplemental-information"><ul class="rlist"><li class="supplemental-information__item" id="ec0005"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/3cce0e77-b5f4-484e-af63-6decc1829c3c/mmc1.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.32
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1</div><div class="supplemental-information__description"><p>Input files for Gan</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="ec0010"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/6b63d7fd-d880-4ec1-888a-4d4cf4f9a589/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (28.62
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2</div><div class="supplemental-information__description"><p>Output files from Gan</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="ec0015"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/02f9901d-9f03-4918-bc5c-fe3c7d6da0b7/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (28.02
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3</div><div class="supplemental-information__description"><p>Names created by combining prefixes and existing species names: first of three files</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="ec0020"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/d723663f-acaf-4363-8a81-39b6b65a2abe/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (34.55
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4</div><div class="supplemental-information__description"><p>Names created by combining prefixes and existing species names: second of three files</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="ec0025"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/dff3af97-699c-4d7c-a5bb-66edd81d0021/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (38.36
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5</div><div class="supplemental-information__description"><p>Names created by combining prefixes and existing species names: third of three files</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="ec0030"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="https://www.cell.com/cms/10.1016/j.tim.2020.10.009/attachment/a32754d2-cb80-4043-81c3-63e7454776fd/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.01
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6</div><div class="supplemental-information__description"><p>Template for Gan input files</p></div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secst0085" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Linnaeus C. </li></ul><div class="ref__title">Systema Naturae.</div>10th edn.  <span class="refPublisherName">Laurentius Salvius</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1759</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0005" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1759&amp;author=Linnaeus+C.&amp;title=Systema+Naturae">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Müller O.F. </li></ul><div class="ref__title">Animalcula infusoria fluviatilia et marina, quae detexit, systematice descripsit et ad vivum delineari curavit.</div> <span class="refPublisherName">Typis Nicolai Mölleri</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1786</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0010" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1786&amp;author=M%C3%BCller+O.F.&amp;title=Animalcula+infusoria+fluviatilia+et+marina%2C+quae+detexit%2C+systematice+descripsit+et+ad+vivum+delineari+curavit">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Darwin C. </li></ul><div class="ref__title">On the Origin of Species.</div>1st edn.  <span class="refPublisherName">John Murray</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1859</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0015" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1859&amp;author=Darwin+C.&amp;title=On+the+Origin+of+Species">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hennig W. </li></ul><div class="ref__title">Grundzüge einer Theorie der phylogenetischen Systematik.</div> <span class="refPublisherName">Deutscher Zentralverlag</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1950</span> <span class="refComment">(in German)</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0020" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1950&amp;author=Hennig+W.&amp;title=Grundz%C3%BCge+einer+Theorie+der+phylogenetischen+Systematik">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parker C.T. </li><li>et al.</li></ul><div class="ref__title">International Code of Nomenclature of Prokaryotes.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">S1-S111</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0025" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85050686253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (99)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=26596770&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.000778&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2019&amp;pages=S1-S111&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Parker+C.T.&amp;title=International+Code+of+Nomenclature+of+Prokaryotes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parte A.C. </li><li>et al.</li></ul><div class="ref__title">List of prokaryotic names with standing in nomenclature (LPSN) moves to the DSMZ.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span> (<span class="refComment">Published online July 23, 2020. https://doi.org/10.1099/ijsem.0.004332</span>)</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0030" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85089371956&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32701423&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.004332&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Parte+A.C.&amp;title=List+of+prokaryotic+names+with+standing+in+nomenclature+%28LPSN%29+moves+to+the+DSMZ">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rees T. </li><li>et al.</li></ul><div class="ref__title">All genera of the world: an overview and estimates based on the March 2020 release of the Interim Register of Marine and Nonmarine Genera (IRMNG).</div><span class="ref__series"><i>Megataxa.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">123-140</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0035" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.11646%2Fmegataxa.1.2.3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2020&amp;pages=123-140&amp;journal=Megataxa&amp;author=Rees+T.&amp;title=All+genera+of+the+world%3A+an+overview+and+estimates+based+on+the+March+2020+release+of+the+Interim+Register+of+Marine+and+Nonmarine+Genera+%28IRMNG%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dykhuizen D.E. </li></ul><div class="ref__title">Santa Rosalia revisited: why are there so many species of bacteria.</div><span class="ref__series"><i>Antonie van Leeuwenhoek.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">25-33</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0040" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0031981808&amp;cf=" onclick="newWindow(this.href);return false">Scopus (215)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=9602276&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1023%2FA%3A1000665216662&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=1998&amp;pages=25-33&amp;journal=Antonie+van+Leeuwenhoek&amp;author=Dykhuizen+D.E.&amp;title=Santa+Rosalia+revisited%3A+why+are+there+so+many+species+of+bacteria">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Amann R. </li><li class="ref__authors__name">Rosselló-Móra R. </li></ul><div class="ref__title">After all, only millions.</div><span class="ref__series"><i>mBio.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>e00999-16<div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0045" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-84986593817&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=27381294&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FmBio.00999-16&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;journal=mBio&amp;author=Amann+R.&amp;author=Rossell%C3%B3-M%C3%B3ra+R.&amp;title=After+all%2C+only+millions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jonson B. </li></ul><div class="ref__title">To the memory of my beloved, The Author Mr William Shakespeare: And what he hath left us.</div>in: Mr. William Shakespeares Comedies Histories &amp; Tragedies: Published according to the True Originall Copies.  <span class="refPublisherName">Isaac Iaggard and Edward Blount</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1623</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0050" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Jonson+B.+To+the+memory+of+my+beloved%2C+The+Author+Mr+William+Shakespeare%3A+And+what+he+hath+left+us.in%3A+Mr.+William+Shakespeares+Comedies+Histories+%26+Tragedies%3A+Published+according+to+the+True+Originall+Copies.+Isaac+Iaggard+and+Edward+Blount%2C%0A+1623">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">MacAdoo T.O. </li></ul><div class="ref__title">Nomenclatural Literacy.</div>in: <span class="editors">Goodfellow M. </span>Handbook of new bacterial systematics.  <span class="refPublisherName">Academic Press</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1993</span>: <span class="ref__seriesPages">339-358</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0055" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=MacAdoo+T.O.+Nomenclatural+Literacy.in%3A+Goodfellow+M.+Handbook+of+new+bacterial+systematics.+Academic+Press%2C%0A+1993%3A+339-358">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Trüper H.G. </li></ul><div class="ref__title">How to name a prokaryote? Etymological considerations, proposals and practical advice in prokaryote nomenclature.</div><span class="ref__series"><i>FEMS Microbiol. Rev.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">231-249</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0060" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=1999&amp;pages=231-249&amp;journal=FEMS+Microbiol.+Rev.&amp;author=Tr%C3%BCper+H.G.&amp;title=How+to+name+a+prokaryote%3F+Etymological+considerations%2C+proposals+and+practical+advice+in+prokaryote+nomenclature">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li></ul><div class="ref__title">How to name new taxa of Archaea and Bacteria.</div>in: <span class="editors">Whitman W.B. </span>Bergey’s Manual of Systematics of Archaea and Bacteria.  <span class="refPublisherName">John Wiley</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2015</span>: <span class="ref__seriesPages">1-24</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0065" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Oren+A.+How+to+name+new+taxa+of+Archaea+and+Bacteria.in%3A+Whitman+W.B.+Bergey%E2%80%99s+Manual+of+Systematics+of+Archaea+and+Bacteria.+John+Wiley%2C%0A+2015%3A+1-24">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anonymous </li></ul><div class="ref__title">Validation of the publication of new names and new combinations previously effectively published outside the IJSB. List no. 17.</div><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1985; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">223-225</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0070" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0021945642&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-35-2-223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=1985&amp;pages=223-225&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Anonymous+&amp;title=Validation+of+the+publication+of+new+names+and+new+combinations+previously+effectively+published+outside+the+IJSB.+List+no.+17">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Marshall B.J. </li><li class="ref__authors__name">Goodwin C.S. </li></ul><div class="ref__title">Revised nomenclature of <em>Campylobacter pyloridis</em>.</div><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1987; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">68</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0075" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0023120054&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-37-1-68&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1987&amp;pages=68&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Marshall+B.J.&amp;author=Goodwin+C.S.&amp;title=Revised+nomenclature+of+Campylobacter+pyloridis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li><li>et al.</li></ul><div class="ref__title">Wanted: microbiologists with basic knowledge of Latin and Greek to join our ‘nomenclature quality control’ team.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">65</span></span>: <span class="ref__seriesPages">3761-3762</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0080" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-84955579461&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=26653899&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.000663&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2015&amp;pages=3761-3762&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Wanted%3A+microbiologists+with+basic+knowledge+of+Latin+and+Greek+to+join+our+%E2%80%98nomenclature+quality+control%E2%80%99+team">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lagier J.C. </li><li>et al.</li></ul><div class="ref__title">Culturing the human microbiota and culturomics.</div><span class="ref__series"><i>Nat. Rev. Microbiol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">540-550</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0085" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85048972988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (156)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=29937540&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41579-018-0041-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2018&amp;pages=540-550&amp;journal=Nat.+Rev.+Microbiol.&amp;author=Lagier+J.C.&amp;title=Culturing+the+human+microbiota+and+culturomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parks D.H. </li><li>et al.</li></ul><div class="ref__title">Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life.</div><span class="ref__series"><i>Nat. Microbiol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">1533-1542</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0090" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85029026923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (369)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=28894102&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41564-017-0012-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2017&amp;pages=1533-1542&amp;journal=Nat.+Microbiol.&amp;author=Parks+D.H.&amp;title=Recovery+of+nearly+8%2C000+metagenome-assembled+genomes+substantially+expands+the+tree+of+life">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parks D.H. </li><li>et al.</li></ul><div class="ref__title">A complete domain-to-species taxonomy for Bacteria and Archaea.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">1079-1086</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0095" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85084211665&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32341564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41587-020-0501-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2020&amp;pages=1079-1086&amp;journal=Nat.+Biotechnol.&amp;author=Parks+D.H.&amp;title=A+complete+domain-to-species+taxonomy+for+Bacteria+and+Archaea">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lagier J.C. </li><li>et al.</li></ul><div class="ref__title">Naming microorganisms: the contribution of the IHU Méditerranée Infection, Marseille, France.</div><span class="ref__series"><i>New Microbes New Infect.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">S89-S95</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85052989723&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=30402249&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1016%2Fj.nmni.2018.08.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=S89-S95&amp;journal=New+Microbes+New+Infect.&amp;author=Lagier+J.C.&amp;title=Naming+microorganisms%3A+the+contribution+of+the+IHU+M%C3%A9diterran%C3%A9e+Infection%2C+Marseille%2C+France">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Konstantinidis K.T. </li><li>et al.</li></ul><div class="ref__title">Uncultivated microbes in need of their own taxonomy.</div><span class="ref__series"><i>ISME J.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2399-2406</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85031803725&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=28731467&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fismej.2017.113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2017&amp;pages=2399-2406&amp;journal=ISME+J.&amp;author=Konstantinidis+K.T.&amp;title=Uncultivated+microbes+in+need+of+their+own+taxonomy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Murray A.E. </li><li>et al.</li></ul><div class="ref__title">Roadmap for naming uncultivated Archaea and Bacteria.</div><span class="ref__series"><i>Nat. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">987-994</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0110" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85086165710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32514073&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1038%2Fs41564-020-0733-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2020&amp;pages=987-994&amp;journal=Nat.+Microbiol.&amp;author=Murray+A.E.&amp;title=Roadmap+for+naming+uncultivated+Archaea+and+Bacteria">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Murray R.G. </li><li class="ref__authors__name">Stackebrandt E. </li></ul><div class="ref__title">Taxonomic note: implementation of the provisional status Candidatus for incompletely described procaryotes.</div><span class="ref__series"><i>Int. J. Syst. Bacteriol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">186-187</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-0028887837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (297)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=7857801&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2F00207713-45-1-186&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1995&amp;pages=186-187&amp;journal=Int.+J.+Syst.+Bacteriol.&amp;author=Murray+R.G.&amp;author=Stackebrandt+E.&amp;title=Taxonomic+note%3A+implementation+of+the+provisional+status+Candidatus+for+incompletely+described+procaryotes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li><li>et al.</li></ul><div class="ref__title">Lists of names of prokaryotic Candidatus taxa.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">3956-4042</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85088879544&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32603289&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.003789&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2020&amp;pages=3956-4042&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Lists+of+names+of+prokaryotic+Candidatus+taxa">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gilroy R. </li><li>et al.</li></ul><div class="ref__title">A genomic blueprint of the chicken gut microbiome.</div><span class="ref__series"><i>Res. Square.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span> (<span class="refComment">Published online August 17, 2020. http://dx.doi.org/10.21203/rs.3.rs-56027/v1</span>)</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;journal=Res.+Square&amp;author=Gilroy+R.&amp;title=A+genomic+blueprint+of+the+chicken+gut+microbiome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dahl R. </li></ul><div class="ref__title">The great automatic grammatizator.</div>in: <em>Someone Like You</em> Alfred.  <span class="refPublisherName">A. Knopf</span>,
        <span class="refPublisherLoc"></span><span class="refDate">1953</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Dahl+R.+The+great+automatic+grammatizator.in%3A+Someone+Like+You+Alfred.+A.+Knopf%2C%0A+1953">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li><li>et al.</li></ul><div class="ref__title">Formation of compound generic names based on personal names: a proposal for emendation of Appendix 9 of the International Code of Nomenclature of Prokaryotes.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">594-596</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85061139837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=30570480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.003192&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2019&amp;pages=594-596&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Formation+of+compound+generic+names+based+on+personal+names%3A+a+proposal+for+emendation+of+Appendix+9+of+the+International+Code+of+Nomenclature+of+Prokaryotes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li></ul><div class="ref__title">Naming novel prokaryotic taxa discovered in the human gut.</div><span class="ref__series"><i>Gut.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">969-970</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85065150710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=31053603&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1136%2Fgutjnl-2019-318716&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2020&amp;pages=969-970&amp;journal=Gut&amp;author=Oren+A.&amp;title=Naming+novel+prokaryotic+taxa+discovered+in+the+human+gut">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pace N.R. </li></ul><div class="ref__title">It’s time to retire the prokaryote.</div><span class="ref__series"><i>Microbiol. Today.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">84</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2009&amp;pages=84&amp;journal=Microbiol.+Today&amp;author=Pace+N.R.&amp;title=It%E2%80%99s+time+to+retire+the+prokaryote">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pace N.R. </li></ul><div class="ref__title">Rebuttal: the modern concept of the procaryote.</div><span class="ref__series"><i>J. Bacteriol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">191</span></span>: <span class="ref__seriesPages">2006-2007</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0150" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-64049096016&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=19168604&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FJB.01296-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=191&amp;publication_year=2009&amp;pages=2006-2007&amp;journal=J.+Bacteriol.&amp;author=Pace+N.R.&amp;title=Rebuttal%3A+the+modern+concept+of+the+procaryote">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pace N.R. </li></ul><div class="ref__title">Problems with 'procaryote'.</div><span class="ref__series"><i>J. Bacteriol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">191</span> (<span class="refComment">discussion 2011</span>)</span>: <span class="ref__seriesPages">2008-2010</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0155" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-64049098525&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=19168605&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1128%2FJB.01224-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=191&amp;publication_year=2009&amp;pages=2008-2010&amp;journal=J.+Bacteriol.&amp;author=Pace+N.R.&amp;title=Problems+with+%27procaryote%27">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hobman J.L. </li><li>et al.</li></ul><div class="ref__title">Laboratory strains of <em>Escherichia coli</em>: model citizens or deceitful delinquents growing old disgracefully.</div><span class="ref__series"><i>Mol. Microbiol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">64</span></span>: <span class="ref__seriesPages">881-885</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0160" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-34248373709&amp;cf=" onclick="newWindow(this.href);return false">Scopus (39)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=17501914&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1111%2Fj.1365-2958.2007.05710.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=2007&amp;pages=881-885&amp;journal=Mol.+Microbiol.&amp;author=Hobman+J.L.&amp;title=Laboratory+strains+of+Escherichia+coli%3A+model+citizens+or+deceitful+delinquents+growing+old+disgracefully">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sutcliffe I.C. </li><li>et al.</li></ul><div class="ref__title">Minutes of the International Committee on Systematics of Prokaryotes online discussion on the proposed use of gene sequences as type for naming of prokaryotes, and outcome of vote.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">4416-4417</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0165" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85088880937&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=32628106&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.004303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2020&amp;pages=4416-4417&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Sutcliffe+I.C.&amp;title=Minutes+of+the+International+Committee+on+Systematics+of+Prokaryotes+online+discussion+on+the+proposed+use+of+gene+sequences+as+type+for+naming+of+prokaryotes%2C+and+outcome+of+vote">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Oren A. </li><li>et al.</li></ul><div class="ref__title">Why are so many effectively published names of prokaryotic taxa never validated.</div><span class="ref__series"><i>Int. J. Syst. Evol. Microbiol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">2125-2129</span><div class="ref__controls"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#back-bb0170" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.tim.2020.10.009&amp;key=2-s2.0-85049891496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.tim.2020.10.009&amp;key=29873629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a href="https://www.cell.com/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.tim.2020.10.009&amp;key=10.1099%2Fijsem.0.002851&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=68&amp;publication_year=2018&amp;pages=2125-2129&amp;journal=Int.+J.+Syst.+Evol.+Microbiol.&amp;author=Oren+A.&amp;title=Why+are+so+many+effectively+published+names+of+prokaryotic+taxa+never+validated">Google Scholar</a></li></ul></div></li></ol></section><section class="glossary" id="gl0005" tabindex="0"><h2 class="top" id="secst0090" data-left-hand-nav="Glossary" tabindex="0"><span class="top__sub"></span><span class="top__text">Glossary</span></h2><strong><strong>Ancient Greek</strong></strong><div class="section-paragraph">(abbreviated Gr.) a classical language, the language of the many celebrated poets, playwrights, and philosophers. Even in ancient times, many Greek words were carried over into Latin and this trend continues in the formation of taxonomic names.</div><strong><strong>Binomial</strong></strong><div class="section-paragraph">a Latinised name for biological species, written in italics and composed of two parts, the first capitalised and identifying the genus, the second identifying the species, for example, <em>Escherichia coli.</em></div><strong><strong><em>Candidatus</em></strong></strong><div class="section-paragraph">a category of name for archaeal or bacterial taxa representing as-yet uncultured organisms. The Code grants no standing to such names, but specifies that they should be prefixed with <em>Candidatus</em> (in italics), the genus and species names should be in Roman type, and the entire name in quotation marks; for example '<em>Candidatus</em> Phytoplasma allocasuarinae'.</div><strong><strong>Cladistics</strong></strong><div class="section-paragraph">an approach to biological classification, pioneered by the German entomologist Willi Hennig, in which organisms are grouped into monophyletic groups (also called clades) and classification strictly reflects phylogeny.</div><strong><strong>Culturomics</strong></strong><div class="section-paragraph">high-throughput culture of microbes as an approach to discover taxonomic novelty.</div><strong><strong>Etymology</strong></strong><div class="section-paragraph">justification for the new name, including a description of constituent terms, their origins, grammatical properties, and meanings.</div><strong><strong>Genome-based taxonomy</strong></strong><div class="section-paragraph">an approach to molecular phylogenetics in which genome sequences are used to create phylogenies and taxonomies: epitomised by the Genome Taxonomy Database (GTDB).</div><strong><strong>International Code of Nomenclature of Prokaryotes</strong></strong><div class="section-paragraph">the ICNP (or ‘the Code’), the set of rules and recommendations for naming Bacteria and Archaea maintained by the International Committee on Systematics of Prokaryotes.</div><strong><strong>Latin</strong></strong><div class="section-paragraph">(abbreviated L.) a classical language, the dominant language of Ancient Rome, which has remained, as New Latin or Neo-Latin (abbreviated N.L.), the language in which taxonomic names are framed.</div><strong><strong>List of Prokaryotic Names with Standing in Nomenclature (LPSN)</strong></strong><div class="section-paragraph">an online database that documents the names of Archaea and Bacteria validly published under the Rules of the ICNP, together with effectively but not validly published names, <em>Candidatus</em> names, and names of cyanobacterial taxa validly published under the Botanical Code.</div><strong><strong>Metagenome-assembled genome (MAG)</strong></strong><div class="section-paragraph">a genome sequence that has been reconstructed by assembling and binning metagenomic reads.</div><strong><strong>Molecular phylogenetics</strong></strong><div class="section-paragraph">analysis of inherited differences in informational macromolecular sequences (DNA, RNA, or proteins) to identify evolutionary relationships and construct phylogenetic trees.</div><strong><strong>Monophyletic group</strong></strong><div class="section-paragraph">another term for a clade, a taxonomic group that includes a common ancestor and all its descendants.</div><strong><strong>Nomenclature</strong></strong><div class="section-paragraph">a system for giving names to organisms.</div><strong><strong><em>Nomina nuda</em></strong></strong><div class="section-paragraph">a term (singular: <em>nomen nudum</em>) for names that look like taxonomic names but have no standing as they have not been published according to the rules of the relevant nomenclatural code.</div><strong><strong>Protologue</strong></strong><div class="section-paragraph">a description of a new taxon which includes the etymology of the name, a description of the taxon, and of the designated nomenclatural type (a strain for a new species, but a species for a new genus).</div><strong><strong>Taxonomy</strong></strong><div class="section-paragraph">the branch of biology concerned with the classification, identification, and nomenclature of organisms; it can also refer to a particular scheme for categorisation.</div></section><section class="article-info"><h2 class="top" id="articleInformation">Article Info</h2><section><h3 class="publicationHistory__label">Publication History</h3><div class="article-info__dates"><div class="article-info__date">Published online: December 04, 2020</div></div></section><section><h3>Publication stage</h3>In Press Corrected Proof</section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.tim.2020.10.009">https://doi.org/10.1016/j.tim.2020.10.009</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2020 The Author(s). Published by Elsevier Ltd.</div></section><section class="userLicenseRpl">                                              <h3>User License</h3>                                               <a href="http://creativecommons.org/licenses/by/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution (CC BY 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="reference-citations__ctrl">How you can reuse</a>                                                     <img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/information.png" alt="Information Icon" class="infoIcon">                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution (CC BY 4.0)</span></div>                                                       <div class="modal-body"><p></p><h2>Permitted</h2>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>
<li>Text &amp; data mine</li>
<li>Translate the article</li>
<li>Reuse portions or extracts from the article in other works</li>
<li>Sell or re-use for commercial purposes</li>
</ul>


<br><a href="https://www.elsevier.com/about/open-science/open-access" target="blank">Elsevier's open access license policy</a><p></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/science/article/pii/S0966842X20302717">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer landscape" style=""><nav class="nav-bar figure-viewer__top"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">The Next Million Names for Archaea and Bacteria</div></nav><div class="frames" style="height: 25809.2px; top: 65px; width: 515.2px;"><ul class="frames__list" style="left: 0px; width: 515.2px;"><li style="height: 25809.2px; left: 0px; width: 515.2px;"><div class="thumbnail__image" style="height: 384.792px; top: 12712.2px; width: 455.2px;"><img class="b-lazy ui-draggable ui-draggable-handle" alt="Figure 1" data-locator="f0005" data-high-res="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" data-large-image="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" tabindex="0" style="position: relative;" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gr1.jpg"></div></li></ul><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Previous figure" class="frameButton previousFrameButton frames__prev" style="left: 0px;"><i class="icon-arrow_l"></i></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Next figure" class="frameButton nextFrameButton frames__next" style="right: 0px;"><i class="icon-arrow_r"></i></a></div><div class="caption" style="left: 515.2px; top: 0px;"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev" style="visibility: hidden;"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next" style="visibility: hidden;"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist" style="width: 134px; left: 0px;"><li><div class="thumbnail__label">Fig. 1</div><div class="thumbnail"><div class="thumbnail__image"><img class="b-lazy" data-src="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" alt="Figure 1" data-locator="f0005" data-high-res="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" data-large-image="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/gr1.jpg" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="https://www.cell.com/action/downloadFigures?pii=S0966-842X(20)30271-7&amp;id=gr1.jpg" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><section><h2 id="figures" class="top">Figures</h2><ul class="figures rlist"><li class="figures__thumbnail"><div class="thumbnail__image thumbnail__image--img-ratio"><div><img class="b-lazy" data-src="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" alt="Figure 1" data-locator="f0005" data-high-res="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/faeb5b09-963b-413a-836d-138cfb13555b/gr1_lrg.jpg" data-large-image="//els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/482b8ba5-63a2-4ce4-af0f-bf5e4bc85fc4/gr1.jpg" tabindex="0"></div></div><div class="figure__title"><span class="label figure__label">Figure 1</span><span class="figure__title__text">Key Figure. Combinatorial Approaches to the Creation of New Genus Names for Archaea and Bacteria.</span></div></li></ul></section><section class="table-list"><h2 id="tables" class="top">Tables</h2><ul class="tabels-list__wrapper rlist"><li class="table-list__item"><a target="_blank" href="https://www.cell.com/action/showFullTableHTML?isHtml=true&amp;tableId=t0005&amp;pii=S0966-842X%2820%2930271-7"><span class="inline-table__label">Table 1</span><span class="inline-table__title">Prefixes Used to Create New Genus or Species Names</span></a></li><li class="table-list__item"><a target="_blank" href="https://www.cell.com/action/showFullTableHTML?isHtml=true&amp;tableId=t0010&amp;pii=S0966-842X%2820%2930271-7"><span class="inline-table__label">Table 2</span><span class="inline-table__title">Final Word Elements for Bacterial or Archaeal Genera<div class="dropBlock reference-citations"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" id="back-tf0005" data-db-target-for="tf0005-d8213092e432" class="reference-citations__ctrl scroll-into-link" aria-expanded="false" aria-controls="tf0005"><sup>a</sup></a><div data-db-target-of="tf0005-d8213092e432" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>a </span><span>Abbreviations: fem., feminine; L., Latin; masc., masculine; neut., neuter; N.L., New Latin.</span></div></div><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></a></li></ul></section>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

            

            

            <div data-widget-id="644a09e2-669e-4c41-afb5-e6aae51a483d" data-widget-name="UX3HTMLWidget"><section>
 <h2 class="top" id="comments-heading">Comments</h2>
<div id="disqus_thread"><iframe id="dsq-app3347" name="dsq-app3347" allowtransparency="true" frameborder="0" scrolling="no" tabindex="0" title="Disqus" width="100%" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/saved_resource.html" style="width: 1px !important; min-width: 100% !important; border: none !important; overflow: hidden !important; height: 391px !important;" horizontalscrolling="no" verticalscrolling="no"></iframe></div>
<script>
var pgUrl = "http://www.cell.com/retrieve/pii/" + "S0966-842X(20)30271-7";
var pgDoi = "10.1016/j.tim.2020.10.009";
var pgTitle = document.getElementsByTagName("title")[0].innerHTML;
var disqus_config = function() {
this.page.url = pgUrl; 
this.page.identifier = pgDoi;
this.page.title = pgTitle;
};
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = 'https://cell-press.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<h4>Cell Press Commenting Guidelines</h4>
<p>To submit a comment for a journal article, please use the space above and note the following:</p>
<ul>
<li>We will review submitted comments within 2 business days.</li>
<li>This forum is intended for constructive dialog. Comments that are commercial or promotional in nature, pertain to specific medical cases, are not relevant to the article for which they have been submitted, or are otherwise inappropriate will not be posted.</li>
<li>We recommend that commenters identify themselves with full names and affiliations.</li>
<li>Comments must be in compliance with our <a href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>.</li>
<li>Comments will not be peer-reviewed.</li>
</ul>
</section></div>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="https://www.cell.com/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="https://www.cell.com/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"><div class="transplanted-clone"></div></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30271-7?rss=yes';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/javascript" id="hs-script-loader" async="" defer="" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd " style="display: none;"><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad gpt-ad-defined" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4" id="gpt-ad-11"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="https://www.cell.com/home" title="Cell Press"><img alt="Cell Press logo" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/CellPress_rebrand_footer.svg"></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
            <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank"><i aria-hidden="true" class="icon-Icon_Twitter"></i></a></li>
            
            <li><a href="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>


<style>

.ux3 .footer__logo img {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Research Journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="https://www.cell.com/cell/home">Cell</a>
		</li>
			<li>
				<a alt="Cancer Cell" href="https://www.cell.com/cancer-cell/home">Cancer Cell</a>
			</li>
			<li>
				<a alt="Cell Chemical Biology" href="https://www.cell.com/cell-chemical-biology">Cell Chemical Biology</a>
			</li>
			<li><a alt="Cell Genomics" href="https://www.cell.com/cell-genomics/home">Cell Genomics</a></li>
			<li>
				<a alt="Cell Host and Microbe" href="https://www.cell.com/cell-host-microbe/home">Cell Host &amp; Microbe</a>
			</li>
			<li>
				<a alt="Cell Metabolism" href="https://www.cell.com/cell-metabolism/home">Cell Metabolism</a>
			</li>
			<li>
				<a alt="Cell Reports" href="https://www.cell.com/cell-reports/home">Cell Reports</a>
			</li>
						<li>
				<a alt="Cell Reports Medicine" href="https://www.cell.com/cell-reports-medicine/home">Cell Reports Medicine</a>
			</li>
			
									<li>
				<a alt="Cell Reports Methods" href="https://www.cell.com/cell-reports-methods/home">Cell Reports Methods</a>
			</li>
			<li>
				<a alt="Cell Reports Physical Science" href="https://www.cell.com/cell-reports-physical-science/home">Cell Reports Physical Science</a>
			</li>
			<li>
				<a alt="Cell Stem Cell" href="https://www.cell.com/cell-stem-cell/home">Cell Stem Cell</a>
			</li>
			<li>
				<a alt="Cell Systems" href="https://www.cell.com/cell-systems/home">Cell Systems</a>
			</li>
			<li>
				<a alt="Chem" href="https://www.cell.com/chem/home">Chem</a></li>
							
			<li>	<a alt="Chem" href="https://www.cell.com/chem-catalysis/home">Chem Catalysis</a>
			</li>
			<li>
				<a alt="Current Biology" href="https://www.cell.com/current-biology/home">Current Biology</a>
			</li>
			<li>
				<a alt="Developmental Cell" href="https://www.cell.com/developmental-cell/home">Developmental Cell</a>
			</li>
			<li>
						<a alt="EBioMedicine" href="https://www.heliyon.com/">Heliyon</a>
		</li>	
			<li>
				<a alt="Immunity" href="https://www.cell.com/immunity/home">Immunity</a>
			</li>
			<li>
				<a alt="iScience" href="https://www.cell.com/iscience/home">iScience</a>
			</li>
			<li>
				<a alt="Joule" href="https://www.cell.com/joule/home">Joule</a>
			</li>
						<li>
				<a alt="Matter" href="https://www.cell.com/matter/home">Matter</a>
			</li>
				<li>
				<a alt="Med" href="https://www.cell.com/med/home">Med</a>
			</li>
			<li>
				<a alt="Molecular Cell" href="https://www.cell.com/molecular-cell/home">Molecular Cell</a>
			</li>
			<li>
				<a alt="Neuron" href="https://www.cell.com/neuron/home">Neuron</a>
			</li>
			<li>
				<a alt="One Earth" href="https://www.cell.com/one-earth/home">One Earth</a>
			</li>
			<li>
				<a alt="Patterns" href="https://www.cell.com/patterns/home">Patterns</a>
			</li>
			<li>
				<a alt="STAR Protocols" href="https://www.cell.com/star-protocols/home">STAR Protocols</a>
			</li>
			<li>
				<a alt="Structure" href="https://www.cell.com/structure/home">Structure</a>
			</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Trends Reviews Journals</div>
		<ul class="rlist clearfix sided">
			<li>
				<a alt="Trends in Biochemical Sciences" href="https://www.cell.com/trends/biochemical-sciences/home">Biochemical Sciences</a>
			</li>
			<li>
				<a alt="Trends in Biotechnology" href="https://www.cell.com/trends/biotechnology/home">Biotechnology</a>
			</li>
			<li>
				<a alt="Trends in Cancer" href="https://www.cell.com/trends/cancer/home">Cancer</a>
			</li>
			<li>
				<a alt="Trends in Cell Biology" href="https://www.cell.com/trends/cell-biology/home">Cell Biology</a>
			</li>
			<li>
				<a alt="Trends in Chemistry" href="https://www.cell.com/trends/chemistry/home">Chemistry</a>
			</li>
			<li>
				<a alt="Trends in Cognitive Sciences" href="https://www.cell.com/trends/cognitive-sciences/home">Cognitive Sciences</a>
			</li>
			<li>
				<a alt="Trends in Ecology and Evolution" href="https://www.cell.com/trends/ecology-evolution/home">Ecology &amp; Evolution</a>
			</li>
			<li>
				<a alt="Trends in Endocrinology and Metabolism" href="https://www.cell.com/trends/endocrinology-metabolism/home">Endocrinology &amp; Metabolism</a>
			</li>
			<li>
				<a alt="Trends in Genetics" href="https://www.cell.com/trends/genetics/home">Genetics</a>
			</li>
			<li>
				<a alt="Trends in Immunology" href="https://www.cell.com/trends/immunology/home">Immunology</a>
			</li>
			<li>
				<a alt="Trends in Microbiology" href="https://www.cell.com/trends/microbiology/home">Microbiology</a>
			</li>
			<li>
				<a alt="Trends in Molecular Medicine" href="https://www.cell.com/trends/molecular-medicine/home">Molecular Medicine</a>
			</li>
			<li>
				<a alt="Trends in Neurosciences" href="https://www.cell.com/trends/neurosciences/home">Neurosciences</a>
			</li>
			<li>
				<a alt="Trends in Parasitology" href="https://www.cell.com/trends/parasitology/home">Parasitology</a>
			</li>
			<li>
				<a alt="Trends in Pharmacological Sciences" href="https://www.cell.com/trends/pharmacological-sciences/home">Pharmacological Sciences</a>
			</li>
			<li>
				<a alt="Trends in Plant Science" href="https://www.cell.com/trends/plant-science/home">Plant Science</a>
			</li>
		</ul>
	</div>

<div class="group">
	<div class="group__title">Partner Journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="American Journal of Human Genetics" href="https://www.cell.com/ajhg/home">AJHG</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="https://www.cell.com/biophysj/home">Biophysical Journal</a>
		</li>
		<li>
			<a alt="EBioMedicine" href="https://www.ebiomedicine.com/">EBioMedicine</a>
		</li>
		<li><a alt="HGG Advances" href="https://www.cell.com/hgg-advances/home">HGG Advances</a></li>
		<li>
			<a alt="Molecular Plant" href="https://www.cell.com/molecular-plant/home">Molecular Plant</a>
		</li>
		<li>
			<a alt="Molecular Therapy Family" href="https://www.cell.com/molecular-therapy-family/home">Molecular Therapy Family</a>
		</li>
				<li>
			<a alt="Plant Communications" href="https://www.cell.com/plant-communications/home">Plant Communications</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="https://www.cell.com/stem-cell-reports/home">Stem Cell Reports</a>
		</li>
        		<li>
			<a alt="The Innovation" href="https://www.cell.com/the-innovation/home">The Innovation</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Collections</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Best Ofs" href="https://www.cell.com/bestof">Best of Cell Press</a>
				</li>
				<li>
					<a alt="Cell Press Reviews" href="https://www.cell.com/reviews">Cell Press Reviews</a>
				</li>
				<li>
					<a alt="Cell Selections" href="https://www.cell.com/selections">Cell Press Selections</a>
				</li>
								<li>
					<a alt="Consortia Hub" href="https://www.cell.com/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Nucleus Collections" href="https://www.cell.com/nucleus">Nucleus Collections</a>
				</li>
				<li>
					<a alt="SnapShots" href="https://www.cell.com/snapshots">SnapShot Archive</a>
				</li>
				<li>
					<a alt="Trends Limited Edition" href="https://www.cell.com/trends/limited-edition">Trends Limited Editions</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">Beyond the Journal</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Career Network" target="_blank" href="http://careers.cell.com/">Cell Career Network</a>
				</li>
				<li>
					<a alt="Cell Mentor" target="_blank" href="https://www.cell.com/cell-mentor">Cell Mentor</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="https://www.cell.com/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Lablinks" href="https://www.cell.com/lablinks">LabLinks</a>
				</li>
				<li>
					<a alt="Webinars" href="https://www.cell.com/webinars">Webinars</a>
				</li>
			</ul>
		</div>
	</div>


	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Evolving the Article</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Figure360" href="https://www.cell.com/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Sneak Peek" href="https://www.cell.com/sneakpeek">Sneak Peek</a>
				</li>
				<li>
					<a alt="STAR Methods" href="https://www.cell.com/star-methods">STAR Methods</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">Science in Society</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Picture Show" href="https://www.cell.com/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Cell Press Podcast" href="https://www.cell.com/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="https://www.cell.com/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="https://www.cell.com/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Research Arc" href="https://www.cell.com/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>
	</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Connect</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="About Cell Press" href="https://www.cell.com/about">About Cell Press</a>
			</li>
						<li>
				<a alt="Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs">Careers</a>
			</li>
			<li>
				<a alt="Contact" href="https://www.cell.com/contact">Contact</a>
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>
			</li>
			<li>
				<a alt="Newsroom" href="https://www.cell.com/newsroom">Newsroom</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="https://www.cell.com/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="https://www.cell.com/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Read It Now" href="https://www.cell.com/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Mobile" href="https://www.cell.com/mobile">Mobile</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content">Recommend to Librarian</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" target="_blank" href="https://media-kit-2019.elsevierdigitaledition.com/">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="http://cellpress.concevoirlabs.com/wp-content/uploads/2018/06/CP-RecruitmentKit-US-2018-LowRes.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" href="https://www.cell.com/librarians">For Librarians</a>
			</li>
			<li>
				<a alt="Cookies" href="https://www.cell.com/cookies">Cookies</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Accessibility" href="https://www.cell.com/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div>
</div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><p class="align-center copy-right">
    <span>We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the <a href="https://www.cell.com/cookies" target="_blank">use of cookies</a>.</span><br>
            Copyright © 2021 Elsevier Inc. except certain content provided by third parties.
<br>
<br>
<a href="https://www.relx.com/"><img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/relx.png" alt="RELX" rel="nofollow" target="_new"></a>
</p></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{background-color:#fff;padding:7px 26px 7px 30px;color:#007DBC;font-size:14px;clear:both;width:100%}#brandingRow a{text-decoration:none;color:#007DBC}#brandingRow a:hover{text-decoration:underline}#brandingRowRight{display:inline;float:right;width:auto;padding:0;margin:0}svg#cp_logo{float:left;height:36px;width:36px;margin-left:16px;margin-top:21px;}@media screen and (max-width: 600px){#brandingRow{font-size:12px}}@media screen and (max-width: 750px){#brandingRow{padding:7px 18px 7px 16px}svg#cp_logo{margin-right:10px}}@media screen and (max-width: 800px){svg#cp_logo{display:none}}.header__wrapper.clearfix .col-xs-6:first-of-type{width:55%}.header__wrapper.clearfix .col-xs-6:nth-of-type(2){width:45%}@media screen and (min-width: 800px) and (max-width: 1000px){.header__wrapper.clearfix .col-xs-6:first-of-type{width:75%}.header__wrapper.clearfix .col-xs-6:nth-of-type(2){width:25%}}</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn") {
	partnerx = true;
	ribbonx = "A Cell Press partner journal";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "A Cell Press journal";
} else {
	ribbonx = "A Cell Press resource";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13	C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2	l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92 	c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8	c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}

</script></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><div id="sso-urls-div" class="hidden ssoUrls">
    
        <iframe src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/update.html" zc3mm4tc3=""></iframe>
    
        <iframe src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/update(1).html" zc3mm4tc3=""></iframe>
    
        <iframe src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/update(2).html" zc3mm4tc3=""></iframe>
    
        <iframe src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/update(3).html" zc3mm4tc3=""></iframe>
    
</div>

<!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        
        </div>
    </div>
































    



        
            
                <script src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/main.bundle-6880250288e388cab83a.js"></script>
            
        
        
        

    






<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0966-842X","name":"Trends in Microbiology"},"page":{"loadTimestamp":"1610964744444","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"95.91.243.99","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0966842X20302717","type":"XOCS-JOURNAL:SCOPE-FULL","title":"The Next Million Names for Archaea and Bacteria","status":"S200"}]};
</script>

<script type="text/javascript">pageDataTracker.trackPageLoad();</script>
<script defer="" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;613789913e820609&quot;,&quot;version&quot;:&quot;2021.1.1&quot;,&quot;si&quot;:10}"></script>


<div id="crossmark-tooltip-130" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>Click to get updates and verify authenticity.</p></div><div class="cmttbot"></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><a target="_blank" href="http://www.crossref.org/crossmark"><img class="crossmark-reset crossmark-popup__logo"></a><button class="crossmark-reset crossmark-popup__btn-close"></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/saved_resource(1).html"></iframe></div></div></div></div></div><img src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/logger.php"><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><iframe style="display: none;" src="./The Next Million Names for Archaea and Bacteria_ Trends in Microbiology_files/saved_resource(2).html"></iframe></body></html>